--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/FacesConfigDigester.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/Digester__TO__SAXBuilder/FacesConfigDigester.java
@@ -1,6 +1,5 @@
-FacesConfigDigester.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 /**
  *  Licensed to the Apache Software Foundation (ASF) under one or more
  *  contributor license agreements.  See the NOTICE file distributed with
@@ -324,5 +323,5 @@
 
 }
 
-<|editable_region_end|>
+
 ```