--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/ForeignMetadataImportServiceBean.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/InputFactory__TO__SAXParser/ForeignMetadataImportServiceBean.java
@@ -1,4 +1,3 @@
-
 package edu.harvard.iq.dataverse.metadataimport;
 
 
@@ -70,7 +69,10 @@
         
         try {
             reader = new StringReader(xmlToParse);
-            XMLInputFactory xmlFactory = XmlUtil.getSecureXMLInputFactory();
+            // Replace StAX (XMLInputFactory) with SAX (SAXParserFactory) for XML parsing
+            javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+            
+            javax.xml.parsers.SAXParser spf = spf.newSAXParser();
             xmlFactory.setProperty(XMLInputFactory.SUPPORT_DTD, false);
             xmlr =  xmlFactory.createXMLStreamReader(reader);
             processXML(xmlr, mappingSupported, datasetVersion);
@@ -98,7 +100,9 @@
         
         try {
             in = new FileInputStream(xmlFile);
-            XMLInputFactory xmlFactory = XmlUtil.getSecureXMLInputFactory();
+            // Replace StAX (XMLInputFactory) with SAX (SAXParserFactory) for XML parsing
+            javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+            javax.xml.parsers.SAXParser spf = spf.newSAXParser();
             xmlr =  xmlFactory.createXMLStreamReader(in);
             processXML(xmlr, mappingSupported, datasetVersion);
         } catch (FileNotFoundException ex) {
@@ -344,4 +348,4 @@
      }
 
     
-}
+}