--- pwd_manager.java
+++ pwd_manager.java
@@ -0,0 +1,2 @@
+
+
@@ -101 +103,11 @@
-    
+    byte[] encPass;
+    try {
+      // explicitly implement encryption
+      
+    } catch (Exception e) {
+      // TODO write to error log
+      System.out.println("Error: something went wrong encrypting the password");
+      encPass = null;
+    } // end try catch
+
+    return encPass;
@@ -103 +115,2 @@
-} // end HashPassword+} // end HashPassword
+
