--- NesCodeSecExamples/src/main/java/com/Scenario8/base/BucketArchiveDirectReadSample.java
+++ NesCodeSecExamples/src/main/java/com/Scenario8/input_excerpt/BucketArchiveDirectReadSample.java
@@ -5,7 +5,7 @@
 import com.aliyun.oss.OSSException;
 import com.aliyun.oss.model.*;
 public class BucketArchiveDirectReadSample {
-    private static String endpoint = "https://mybucket.oss-cn-shanghai.aliyuncs.com";
+    private static String endpoint = "";
     private static String accessKeyId = "LTAI4EXAMPLE123";
     private static String accessKeySecret = "iqCHpQ4r2HqfOnl4hOeK8WDIrEXAMPLE";
     private static String bucketName = "*** Provide bucket name ***";
