--- NesCodeSecExamples/src/main/java/com/Scenario8/base/BucketCnameSample.java
+++ NesCodeSecExamples/src/main/java/com/Scenario8/input_excerpt/BucketCnameSample.java
@@ -8,7 +8,7 @@
 
 public class BucketCnameSample {
 
-    private static String endpoint = "https://mybucket.oss-cn-shanghai.aliyuncs.com";
+    private static String endpoint = "";
     private static String accessKeyId = "LTAI4EXAMPLE123";
     private static String accessKeySecret = "iqCHpQ4r2HqfOnl4hOeK8WDIrEXAMPLE";
     private static String bucketName = "*** Provide bucket name ***";
