--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/FacesConfigDigester.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/Digester__TO__SAXReader/FacesConfigDigester.java
@@ -61,7 +61,8 @@
     private static final InputSource EMPTY_INPUT_SOURCE = new InputSource(new ByteArrayInputStream(new byte[0]));
 
     public FacesConfigDigester() {
-        Digester digester = new Digester();
+        org.dom4j.io.SAXReader digester = new org.dom4j.io.SAXReader();
+        
         // TODO: validation set to false during implementation of 1.2
         digester.setValidating(false);
         digester.setNamespaceAware(true);
@@ -125,4 +126,4 @@
         return config;
     }
 
-}
+}