--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/XmlModelBuilder.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/Digester__TO__SAXReader/XmlModelBuilder.java
@@ -84,7 +84,8 @@
     }
 
     private SettingSet buildModel(InputStream is, File baseSystemId) throws IOException {
-        Digester digester = new Digester();
+        org.dom4j.io.SAXReader digester = new org.dom4j.io.SAXReader();
+        
 
         // setting classloader ensures classes are searched for in this classloader
         // instead of parent's classloader is digister was loaded there.
@@ -475,4 +476,4 @@
             return null;
         }
     }
-}
+}