--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/HusacctResource.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXBuilder__TO__SAXReader/HusacctResource.java
@@ -17,7 +17,8 @@
 	@Override
 	public Document load(HashMap<String, Object> dataValues) {
 		File file = (File) dataValues.get("file");
-		SAXBuilder sax = new SAXBuilder();
+		org.dom4j.io.SAXReader sax = new org.dom4j.io.SAXReader();
+		
 		Document doc = new Document();
 		try {
 			doc = sax.build(file);
@@ -52,4 +53,4 @@
 		return false;
 	}
 
-}
+}