--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/PluginTestUsername.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXBuilder__TO__SAXReader/PluginTestUsername.java
@@ -47,7 +47,8 @@
 	@Before
 	public void setUp() throws Exception {
 
-		SAXBuilder builder = new SAXBuilder();
+		org.dom4j.io.SAXReader builder = new org.dom4j.io.SAXReader();
+
 		try {
 			document = builder.build(new File("./testData.xml")
 					.getCanonicalPath());
@@ -193,4 +194,4 @@
 										mainViewId));
 	}
 
-}
+}