--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/CitygmlPackFile.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXParser__TO__SAXBuilder/CitygmlPackFile.java
@@ -31,2 +31 @@
-        SAXParserFactory factory = SAXParserFactory.newInstance();
-        factory.setValidating(false);
+        org.jdom2.input.SAXBuilder parser = new org.jdom2.input.SAXBuilder();
@@ -34 +32,0 @@
-        SAXParser parser = factory.newSAXParser();
@@ -39 +37 @@
-}
+}