--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/CitygmlPackFile.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXParser__TO__SAXBuilder/CitygmlPackFile.java
@@ -28,12 +28,10 @@
      * 
      */
     public void parse() throws ParserConfigurationException, SAXException, IOException {
-        SAXParserFactory factory = SAXParserFactory.newInstance();
-        factory.setValidating(false);
+        org.jdom2.input.SAXBuilder parser = new org.jdom2.input.SAXBuilder();
         
-        SAXParser parser = factory.newSAXParser();
         try {
 			parser.parse(this, gml);
 		} catch (SAXParseException e) {}
     }
-}
+}