--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/XmlIO.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXParser__TO__SAXBuilder/XmlIO.java
@@ -32,2 +32,2 @@
-            SAXParserFactory saxParserFactory = SAXParserFactory.newInstance();
-            SAXParser saxparser = saxParserFactory.newSAXParser();
+            org.jdom2.input.SAXBuilder saxparser = new org.jdom2.input.SAXBuilder();
+            
@@ -112 +112 @@
-}
+}