--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/XmlIO.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXParser__TO__SAXBuilder/XmlIO.java
@@ -29,8 +29,8 @@
     public static ArrayList<DimenItem> readDimenFile(String baseDimenFilePath) {
         ArrayList<DimenItem> list = null;
         try {
-            SAXParserFactory saxParserFactory = SAXParserFactory.newInstance();
-            SAXParser saxparser = saxParserFactory.newSAXParser();
+            org.jdom2.input.SAXBuilder saxparser = new org.jdom2.input.SAXBuilder();
+            
             InputStream inputStream = new FileInputStream(baseDimenFilePath);
             SAXReadHandler saxReadHandler = new SAXReadHandler();
             saxparser.parse(inputStream, saxReadHandler);
@@ -109,4 +109,4 @@
             e.printStackTrace();
         }
     }
-}
+}