--- NesCodeSecExamples/src/main/java/com/Scenario1/base/DocumentBuilder/ItemProcessor.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/DocumentBuilder__TO__SAXBuilder/ItemProcessor.java
@@ -102,2 +102,2 @@
-        DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
-        DocumentBuilder db = dbf.newDocumentBuilder();
+        org.jdom2.input.SAXBuilder db = new org.jdom2.input.SAXBuilder();
+
@@ -424 +424 @@
-}
+}