--- NesCodeSecExamples/src/main/java/com/Scenario1/base/DocumentBuilder/ItemProcessor.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/DocumentBuilder__TO__SAXBuilder/ItemProcessor.java
@@ -99,8 +99,8 @@
         logger.info("Executing Item Processor.");
         ArrayList<HashMap> listOfUploadFilePaths =container.getListOfUploadFilePaths();        
 
-        DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
-        DocumentBuilder db = dbf.newDocumentBuilder();
+        org.jdom2.input.SAXBuilder db = new org.jdom2.input.SAXBuilder();
+
         InputSource is = new InputSource();
         is.setCharacterStream(new StringReader(container.getRequestBody()));
         Document doc = db.parse(is);
@@ -421,4 +421,4 @@
             dnItemDataMapDao.saveOrUpdate(mapping);
         }
     }
-}
+}