--- NesCodeSecExamples/src/main/java/com/Scenario1/base/DocumentBuilder/XmlBeanDefinitionReader.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/DocumentBuilder__TO__SAXBuilder/XmlBeanDefinitionReader.java
@@ -33,8 +33,8 @@
 	}
 
 	private void doLoadBeanDefinitions(InputStream inputStream) throws Exception {
-		DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
-		DocumentBuilder docBuilder = factory.newDocumentBuilder();
+		org.jdom2.input.SAXBuilder docBuilder = new org.jdom2.input.SAXBuilder();
+		
 		Document doc = docBuilder.parse(inputStream);
 		//解析bean
 		registerBeanDefinitions(doc);
@@ -96,4 +96,4 @@
 	}	
 	
 	
-}
+}