--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/Listing_15_16_StructuredContent.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXParser__TO__Digester/Listing_15_16_StructuredContent.java
@@ -55 +55,2 @@
-        SAXParser parser = SAXParserFactory.newInstance().newSAXParser();
+        org.apache.commons.digester3.Digester parser = new org.apache.commons.digester3.Digester();
+
@@ -65 +66 @@
-}
+}