--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/XmlIO.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXParser__TO__Digester/XmlIO.java
@@ -1,6 +1,5 @@
-XmlIO.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package com.duke.phonescreenmatch_test.dp;
 
 import org.xml.sax.helpers.AttributesImpl;
@@ -32,8 +31,8 @@
     public static ArrayList<DimenItem> readDimenFile(String baseDimenFilePath) {
         ArrayList<DimenItem> list = null;
         try {
-            SAXParserFactory saxParserFactory = SAXParserFactory.newInstance();
-            SAXParser saxparser = saxParserFactory.newSAXParser();
+            org.apache.commons.digester3.Digester saxparser = new org.apache.commons.digester3.Digester();
+            
             InputStream inputStream = new FileInputStream(baseDimenFilePath);
             SAXReadHandler saxReadHandler = new SAXReadHandler();
             saxparser.parse(inputStream, saxReadHandler);
@@ -114,5 +113,5 @@
     }
 }
 
-<|editable_region_end|>
+
 ```