--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/Listing_15_16_StructuredContent.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXParser__TO__SAXReader/Listing_15_16_StructuredContent.java
@@ -1,6 +1,5 @@
-Listing_15_16_StructuredContent.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package com.itextpdf.samples.book.part4.chapter15;
 
 import com.itextpdf.kernel.geom.PageSize;
@@ -55,7 +54,8 @@
         // create a new tag, which will be a kid of the root element, and move to it
         autoTaggingPointer.addTag("chapter");
 
-        SAXParser parser = SAXParserFactory.newInstance().newSAXParser();
+        org.dom4j.io.SAXReader parser = new org.dom4j.io.SAXReader();
+
         List<String> roles = new ArrayList<String>();
         parser.parse(
                 new InputSource(new FileInputStream(RESOURCE)),
@@ -67,5 +67,5 @@
     }
 }
 
-<|editable_region_end|>
+
 ```