--- NesCodeSecExamples/src/main/java/com/Scenario1/base/DocumentBuilder/DOMSearch.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/DocumentBuilder__TO__SAXReader/DOMSearch.java
@@ -1,6 +1,5 @@
-DOMSearch.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 import java.io.IOException;
 
 import java.util.ArrayList;
@@ -24,8 +23,8 @@
    {
       try
       {
-         DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
-         DocumentBuilder db = dbf.newDocumentBuilder();
+         org.dom4j.io.SAXReader db = new org.dom4j.io.SAXReader();
+         
          Document doc = db.parse("contacts.xml");
          List<String> contactNames = new ArrayList<String>();
          NodeList contacts = doc.getElementsByTagName("contact");
@@ -79,5 +78,5 @@
       }
    }
 }
-<|editable_region_end|>
+
 ```