--- NesCodeSecExamples/src/main/java/com/Scenario1/base/DocumentBuilder/XmlBeanDefinitionReader.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/DocumentBuilder__TO__SAXReader/XmlBeanDefinitionReader.java
@@ -36,2 +36,2 @@
-		DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
-		DocumentBuilder docBuilder = factory.newDocumentBuilder();
+		org.dom4j.io.SAXReader docBuilder = new org.dom4j.io.SAXReader();
+		
@@ -99 +99 @@
-}
+}