--- DownloadSample.java
+++ DownloadSample.java
@@ -0,0 +1,2 @@
+
+
@@ -21,4 +23,4 @@
-    private static String accessKeySecret = "";
-    private static String bucketName = "";
-    private static String key = "";
-    private static String downloadFile = "";
+    private static String accessKeySecret = 
+    private static String bucketName = "bucket-name2";
+    private static String key = "kalen-bneric";
+    private static String downloadFile = "download-file.txt";
@@ -67 +69,2 @@
-}+}
+
