--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/ConvertXml2PdfServiceImpl.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXReader__TO__SAXParser/ConvertXml2PdfServiceImpl.java
@@ -48 +48,3 @@
-        SAXReader reader = new SAXReader();
+        javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+        
+        javax.xml.parsers.SAXParser reader = spf.newSAXParser();
@@ -77 +79,2 @@
-        SAXReader reader = new SAXReader();
+        javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+        javax.xml.parsers.SAXParser reader = spf.newSAXParser();
@@ -156 +159 @@
-}
+}