--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/DocumentUtilities.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXReader__TO__SAXParser/DocumentUtilities.java
@@ -49,7 +49,9 @@
 
   public static Document read(String source) throws AnathemaException {
     try {
-      SAXReader saxReader = new SAXReader();
+      javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+      
+      javax.xml.parsers.SAXParser saxReader = spf.newSAXParser();
       return saxReader.read(new StringReader(source));
     } catch (DocumentException exception) {
       throw new AnathemaException(exception);
@@ -66,7 +68,8 @@
 
   public static Document read(InputStream in) throws PersistenceException {
     try {
-      SAXReader saxReader = new SAXReader();
+      javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+      javax.xml.parsers.SAXParser saxReader = spf.newSAXParser();
       return saxReader.read(in, null);
     } catch (DocumentException exception) {
       throw new PersistenceException(exception);
