--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/ErrorFilter.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXReader__TO__SAXParser/ErrorFilter.java
@@ -104,7 +104,9 @@
 
         // JAXB will expect a namespace if we try to unmarshall, but some error responses don't include
         // a namespace. In lieu of writing a SAXFilter to apply a default namespace in-line, this works just as well.
-        SAXReader saxReader = new SAXReader();
+        javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+        
+        javax.xml.parsers.SAXParser saxReader = spf.newSAXParser();
 
         Document d;
         try {
@@ -133,4 +135,4 @@
         log.debug("Error: {}, message: {}, requestId: {}", new Object[]{code, message, requestId});
         return new S3Exception(message, statusCode, code, requestId);
     }
-}
+}