--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/LineMaker.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXReader__TO__SAXParser/LineMaker.java
@@ -71,7 +71,10 @@
             strRoot = strRoot.replaceAll("\\\\", "/");
             String strPath = strRoot + "/conf/";
 
-            SAXReader reader = new SAXReader();
+            javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+
+
+            javax.xml.parsers.SAXParser reader = spf.newSAXParser();
             Document document = reader.read(new File(strPath + "CustomLine.xml"));
             Element elmRoot = document.getRootElement();
             Element elmField = elmRoot.element(strName);
@@ -264,4 +267,4 @@
         listCommand.add(strOutputFile);
     }
 
-}
+}