--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/XmlParseService.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXReader__TO__SAXParser/XmlParseService.java
@@ -65,7 +65,10 @@
 
     public void doXMLParse(String xml,String pageType) throws Exception {
 
-        SAXReader reader = new SAXReader();
+        javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+
+
+        javax.xml.parsers.SAXParser reader = spf.newSAXParser();
         Document document = reader.read(new ByteArrayInputStream(xml.getBytes()));
         Element element = document.getRootElement();
 
@@ -179,6 +182,4 @@
         }
     }
 
-}
-
-    
+}