--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/XmlParseService.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXReader__TO__SAXParser/XmlParseService.java
@@ -68 +68,4 @@
-        SAXReader reader = new SAXReader();
+        javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+
+
+        javax.xml.parsers.SAXParser reader = spf.newSAXParser();
@@ -182,3 +185 @@
-}
-
-    
+}