--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/DefaultSAXPrologProcessRegistry.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXParser__TO__SAXReader/DefaultSAXPrologProcessRegistry.java
@@ -42,7 +42,8 @@
 		//Element cpElement;
 
 		try {
-			SAXParser parser = SAXParserFactory.newInstance().newSAXParser();
+			org.dom4j.io.SAXReader parser = new org.dom4j.io.SAXReader();
+			
 			//parser.setProperty("http://xml.org/sax/features/namespaces", new Boolean(true));
 			//boolean b = parser.isNamespaceAware();
 			parser.parse(new InputSource(reader),new RegistryHandler());
@@ -141,6 +142,4 @@
 
 	
 
-}
-
-
+}