--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/LabsdbService.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXParser__TO__SAXReader/LabsdbService.java
@@ -199,7 +199,8 @@
          }
        }
       //generatorWiremapData(servers,pduIDListMap,networkIDListMap,labsdbClient);
-      SAXParserFactory spf = SAXParserFactory.newInstance();
+      org.dom4j.io.SAXReader spf = new org.dom4j.io.SAXReader();
+      
       WiremapSaxHandler handler = new WiremapSaxHandler(wirmMap_node);
       SAXParser parser = null;
       try {
@@ -335,4 +336,4 @@
       return assetNameAndIdMap;
    }
 
-}
+}