--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/ReadXMLFileSAX.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXParser__TO__SAXReader/ReadXMLFileSAX.java
@@ -13,8 +13,8 @@
 public class ReadXMLFileSAX {
     public static Object readXML(String xmlFile) {
         try {
-            SAXParserFactory factory = SAXParserFactory.newInstance();
-            SAXParser saxParser = factory.newSAXParser();
+            org.dom4j.io.SAXReader saxParser = new org.dom4j.io.SAXReader();
+            
 
             //LogSaxHandler handler = new LogSaxHandler();
             SaxHandler handler = new SaxHandler();
@@ -30,4 +30,4 @@
     }
 
 
-}
+}