--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/ReadXMLFileSAX.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXParser__TO__SAXReader/ReadXMLFileSAX.java
@@ -1,6 +1,5 @@
-ReadXMLFileSAX.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package sax;
 
 import javax.xml.parsers.SAXParser;
@@ -16,8 +15,8 @@
 public class ReadXMLFileSAX {
     public static Object readXML(String xmlFile) {
         try {
-            SAXParserFactory factory = SAXParserFactory.newInstance();
-            SAXParser saxParser = factory.newSAXParser();
+            org.dom4j.io.SAXReader saxParser = new org.dom4j.io.SAXReader();
+            
 
             //LogSaxHandler handler = new LogSaxHandler();
             SaxHandler handler = new SaxHandler();
@@ -35,5 +34,5 @@
 
 }
 
-<|editable_region_end|>
+
 ```