--- NesCodeSecExamples/src/main/java/com/Scenario8/SecureToHardcode/base/PaystackPaymentProcessor.java
+++ NesCodeSecExamples/src/main/java/com/Scenario8/SecureToHardcode/base/PaystackPaymentProcessor.java
@@ -5,7 +5,7 @@
 public class PaystackPaymentProcessor {
 
     private static final String BASE_URL = "https://api.paystack.co";
-    private static final String API_KEY = System.getenv("PAYSTACK_API_KEY");
+    private static final String API_KEY = System.getenv("PAYSTACK_API_KEY")
     private final OkHttpClient client;
     private final Gson gson;
 
