--- NesCodeSecExamples/src/main/java/com/Scenario1/base/DocumentBuilder/DomToGroovy.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/DocumentBuilder__TO__Digester/DomToGroovy.java
@@ -109,16 +109,13 @@
     }
 
     public static Document parse(final Reader input) throws Exception {
-        DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
-        factory.setNamespaceAware(true);
-        DocumentBuilder builder = factory.newDocumentBuilder();
+        org.apache.commons.digester3.Digester builder = new org.apache.commons.digester3.Digester();
+        
         return builder.parse(new InputSource(input));
     }
 
     public static Document parse(final InputStream input) throws Exception {
-        DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
-        factory.setNamespaceAware(true);
-        DocumentBuilder builder = factory.newDocumentBuilder();
+        org.apache.commons.digester3.Digester builder = new org.apache.commons.digester3.Digester();
         return builder.parse(new InputSource(input));
     }
 
@@ -396,4 +393,4 @@
     protected void printIndent() {
         out.printIndent();
     }
-}
+}