--- NesCodeSecExamples/src/main/java/com/Scenario1/base/DocumentBuilder/EarthquakeUpdateJobService.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/DocumentBuilder__TO__Digester/EarthquakeUpdateJobService.java
@@ -102,9 +102,8 @@
       int responseCode = httpConnection.getResponseCode();
       if (responseCode == HttpURLConnection.HTTP_OK) {
         InputStream in = httpConnection.getInputStream();
-        DocumentBuilderFactory dbf =
-          DocumentBuilderFactory.newInstance();
-        DocumentBuilder db = dbf.newDocumentBuilder();
+        org.apache.commons.digester3.Digester db = new org.apache.commons.digester3.Digester();
+        
 
         // Parse the earthquake feed.
         Document dom = db.parse(in);
