--- NesCodeSecExamples/src/main/java/com/Scenario1/base/DocumentBuilder/EarthquakeUpdateJobService.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/DocumentBuilder__TO__Digester/EarthquakeUpdateJobService.java
@@ -105,3 +105,2 @@
-        DocumentBuilderFactory dbf =
-          DocumentBuilderFactory.newInstance();
-        DocumentBuilder db = dbf.newDocumentBuilder();
+        org.apache.commons.digester3.Digester db = new org.apache.commons.digester3.Digester();
+        
