--- NesCodeSecExamples/src/main/java/com/Scenario1/base/DocumentBuilder/EvaluateOsiris.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/DocumentBuilder__TO__Digester/EvaluateOsiris.java
@@ -68,13 +68,8 @@
 		dbSNP.init(mysql, property.getProperty("database.PSM"), property.getProperty("database.hgvs_view"));
 		UniprotFeature.init(mysql, property.getProperty("database.uniprot"));
 
-		DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
-		factory.setFeature("http://xml.org/sax/features/namespaces", false);
-		factory.setNamespaceAware(true);
-		factory.setFeature("http://xml.org/sax/features/validation", false);
-		factory.setFeature("http://apache.org/xml/features/nonvalidating/load-dtd-grammar", false);
-		factory.setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false);
-		DocumentBuilder builder = factory.newDocumentBuilder();
+		org.apache.commons.digester3.Digester builder = new org.apache.commons.digester3.Digester();
+
 
 		XPath xPath = XPathFactory.newInstance().newXPath();
 		XPathExpression documentExp = xPath.compile("/Articles/Article");
@@ -199,4 +194,4 @@
 		System.err.println("F1 " +df.format(f1));
 	}
 
-}
+}