--- NesCodeSecExamples/src/main/java/com/Scenario1/base/DocumentBuilder/XxeDocumentBuilderFactory(2).java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/DocumentBuilder__TO__Digester/XxeDocumentBuilderFactory(2).java
@@ -17,7 +17,8 @@
 public class XxeDocumentBuilderFactory {
 
     public void negative() throws SAXException, DocumentException, ParserConfigurationException, IOException {
-        DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+        org.apache.commons.digester3.Digester factory = new org.apache.commons.digester3.Digester();
+        
         <fim_suffix>
         Document document = builder.parse(in);
     }
