--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/JDomParser.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXBuilder__TO__SAXReader/JDomParser.java
@@ -1,6 +1,5 @@
-JDomParser.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package com.baeldung.xml;
 
 import java.io.File;
@@ -25,7 +24,8 @@
 
     public List<Element> getAllTitles() {
         try {
-            SAXBuilder builder = new SAXBuilder();
+            org.dom4j.io.SAXReader builder = new org.dom4j.io.SAXReader();
+            
             Document doc = builder.build(this.getFile());
             Element tutorials = doc.getRootElement();
             List<Element> titles = tutorials.getChildren("tutorial");
@@ -39,7 +39,7 @@
 
     public Element getNodeById(String id) {
         try {
-            SAXBuilder builder = new SAXBuilder();
+            org.dom4j.io.SAXReader builder = new org.dom4j.io.SAXReader();
             Document document = (Document) builder.build(file);
             String filter = "//*[@tutId='" + id + "']";
             XPathFactory xFactory = XPathFactory.instance();
@@ -63,5 +63,5 @@
 
 }
 
-<|editable_region_end|>
+
 ```