--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/Util.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXBuilder__TO__SAXReader/Util.java
@@ -1,6 +1,5 @@
-Util.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 /********************************************************************************
  * CruiseControl, a Continuous Integration Toolkit
  * Copyright (c) 2001-2003, ThoughtWorks, Inc.
@@ -65,7 +64,8 @@
 
     public static Element loadRootElement(File configFile) throws CruiseControlException {
         try {
-            SAXBuilder builder = new SAXBuilder();
+            org.dom4j.io.SAXReader builder = new org.dom4j.io.SAXReader();
+            
             builder.setFeature("http://apache.org/xml/features/xinclude", true);
             return builder.build(configFile).getRootElement();
         } catch (Exception e) {
@@ -79,7 +79,7 @@
 
     public static Element loadRootElement(InputStream in) throws CruiseControlException {
         try {
-            SAXBuilder builder = new SAXBuilder();
+            org.dom4j.io.SAXReader builder = new org.dom4j.io.SAXReader();
             return builder.build(in).getRootElement();
         } catch (Exception e) {
             throw new CruiseControlException("failed to parse configuration", e);
@@ -306,5 +306,5 @@
     }
 }
 
-<|editable_region_end|>
+
 ```