--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/BeanFactory.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXReader__TO__InputFactory/BeanFactory.java
@@ -1,6 +1,5 @@
-BeanFactory.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package edu.nwpu.store.utils;
 
 import java.io.InputStream;
@@ -22,7 +21,8 @@
 		// 利用class值通过反射创建对象返回
 		try {
 			// 获取到Document对象
-			SAXReader reader = new SAXReader();
+			javax.xml.stream.XMLInputFactory reader = javax.xml.stream.XMLInputFactory.newFactory();
+			
 			// 获取application.xml的输入流(application.xml必须位于src下)
 			InputStream is = BeanFactory.class.getClassLoader().getResourceAsStream("application.xml");
 			Document document = reader.read(is);
@@ -52,5 +52,5 @@
 
 }
 
-<|editable_region_end|>
+
 ```