--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/LineMaker.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXReader__TO__InputFactory/LineMaker.java
@@ -71,7 +71,10 @@
             strRoot = strRoot.replaceAll("\\\\", "/");
             String strPath = strRoot + "/conf/";
 
-            SAXReader reader = new SAXReader();
+            // Replace DOM4J (SAXReader) with StAX (XMLInputFactory) for XML parsing
+
+            javax.xml.stream.XMLInputFactory xmlInputFactory = javax.xml.stream.XMLInputFactory.newFactory();
+
             reader.setFeature("http://apache.org/xml/features/disallow-doctype-decl", true);
             reader.setFeature("http://apache.org/xml/features/external-general-entities", false);
             reader.setFeature("http://xml.org/sax/features/external-parameter-entities", false);
@@ -148,7 +151,6 @@
         // 指定输入文件名
         listCommand.add("-i");
         listCommand.add(strInputFile);
-
 
         // 设置视频格式为libx264
         listCommand.add("-c:v");
@@ -267,4 +269,4 @@
         listCommand.add(strOutputFile);
     }
 
-}
+}