--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/PRConfig.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXReader__TO__InputFactory/PRConfig.java
@@ -1,4 +1,3 @@
-
 package com.pega.gcs.logviewer.systemstate.model;
 
 import java.io.StringReader;
@@ -100,7 +99,10 @@
 
         if (xmlContent != null) {
 
-            SAXReader saxReader = new SAXReader();
+            // Replace DOM4J (SAXReader) with StAX (XMLInputFactory) for XML parsing
+
+            javax.xml.stream.XMLInputFactory xmlInputFactory = javax.xml.stream.XMLInputFactory.newFactory();
+
 
             StringReader stringReader = new StringReader(xmlContent);
 
@@ -125,4 +127,4 @@
         }
 
     }
-}
+}