--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/ErrorFilter.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXReader__TO__Digester/ErrorFilter.java
@@ -1,6 +1,5 @@
-ErrorFilter.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 /*
  * Copyright (c) 2015-2016, EMC Corporation.
  * Redistribution and use in source and binary forms, with or without modification,
@@ -107,7 +106,8 @@
 
         // JAXB will expect a namespace if we try to unmarshall, but some error responses don't include
         // a namespace. In lieu of writing a SAXFilter to apply a default namespace in-line, this works just as well.
-        SAXReader saxReader = new SAXReader();
+        org.apache.commons.digester3.Digester saxReader = new org.apache.commons.digester3.Digester();
+        
 
         Document d;
         try {
@@ -138,5 +138,5 @@
     }
 }
 
-<|editable_region_end|>
+
 ```