--- NesCodeSecExamples/src/main/java/com/Scenario8/base/Client_fr.java
+++ NesCodeSecExamples/src/main/java/com/Scenario8/input_event/chatbot/Client_fr.java
@@ -36,7 +36,7 @@
 public class Client_fr {
 
     static final String REST_URL = "http://bioportal.lirmm.fr:8082/ontologies?&include=all";
-    static final String API_KEY = "759c2da4-f6e1-4b0b-94e9-842a2ab09d01";
+    static final String API_KEY = 
     static final ObjectMapper mapper = new ObjectMapper();
     static HashMap<String , Terme>Termes = new HashMap<String, Terme>();
     static Logger logRoot = Logger.getRootLogger();
