--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/FacesConfigDigester.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/Digester__TO__SAXParser/FacesConfigDigester.java
@@ -61,7 +61,9 @@
     private static final InputSource EMPTY_INPUT_SOURCE = new InputSource(new ByteArrayInputStream(new byte[0]));
 
     public FacesConfigDigester() {
-        Digester digester = new Digester();
+        javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+        
+        javax.xml.parsers.SAXParser digester = spf.newSAXParser();
         // TODO: validation set to false during implementation of 1.2
         digester.setValidating(false);
         digester.setNamespaceAware(true);
@@ -319,4 +321,4 @@
         return config;
     }
 
-}
+}