--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/WebXmlParser.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/Digester__TO__SAXParser/WebXmlParser.java
@@ -137,7 +137,9 @@
 
    private Digester getConfiguredDigester()
    {
-      final Digester digester = new Digester();
+      javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+      
+      javax.xml.parsers.SAXParser digester = spf.newSAXParser();
 
       /*
        * We use the context class loader to resolve classes. This fixes
@@ -174,4 +176,4 @@
       }
       return "";
    }
-}
+}