--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/XMLTextConceptDigester.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/Digester__TO__SAXParser/XMLTextConceptDigester.java
@@ -34,7 +34,10 @@
   public Results parseAlchemyXML(InputStream xmlReader) throws IOException, SAXException,
           URISyntaxException {
 
-    Digester digester = new Digester();
+    javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+
+
+    javax.xml.parsers.SAXParser digester = spf.newSAXParser();
     digester.setValidating(false);
 
     digester.addObjectCreate("results", ConceptResults.class);
@@ -51,4 +54,4 @@
 
     return (Results) digester.parse(xmlReader);
   }
-}
+}