--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/ItspAccounts.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/Digester__TO__SAXParser/ItspAccounts.java
@@ -64 +64,3 @@
-        Digester digester = new Digester();
+        javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+        
+        javax.xml.parsers.SAXParser digester = spf.newSAXParser();
@@ -81 +83 @@
-}
+}