--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/hwxml.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__SAXBuilder/hwxml.java
@@ -1,6 +1,5 @@
-hwxml.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 /**
  * EGroupware - Notifications Java Desktop App
  *
@@ -86,7 +85,8 @@
     {
         if( !this.newfile )
         {
-            XMLInputFactory inputFactory = XMLInputFactory.newInstance();
+            org.jdom2.input.SAXBuilder inputFactory = new org.jdom2.input.SAXBuilder();
+            
 			
 			InputStream in = null;
 			
@@ -178,5 +178,5 @@
     }
 }
 
-<|editable_region_end|>
+
 ```