--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/StatDefManager.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__SAXBuilder/StatDefManager.java
@@ -1,6 +1,5 @@
-StatDefManager.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 /*
  * Copyright 2015, Yahoo Inc.
  * Copyrights licensed under the Apache License.
@@ -130,5 +129,5 @@
 	
 }
 
-<|editable_region_end|>
+
 ```