--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/StaxParser(2).java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__SAXBuilder/StaxParser(2).java
@@ -1,6 +1,5 @@
-StaxParser(2).java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package com.github.mfatihercik.dsb.xml;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
@@ -33,7 +32,8 @@
 
     @Override
     protected void parseInputStream(Reader reader) {
-        XMLInputFactory factory = XMLInputFactory.newInstance();
+        org.jdom2.input.SAXBuilder factory = new org.jdom2.input.SAXBuilder();
+        
         try {
             XMLStreamReader streamReader = factory.createXMLStreamReader(reader);
             processReader(streamReader);
@@ -46,7 +46,7 @@
 
     @Override
     protected void parseInputStream(InputStream inputStream) {
-        XMLInputFactory factory = XMLInputFactory.newInstance();
+        org.jdom2.input.SAXBuilder factory = new org.jdom2.input.SAXBuilder();
         try {
             XMLStreamReader streamReader = factory.createXMLStreamReader(inputStream);
             processReader(streamReader);
@@ -213,5 +213,5 @@
 
 }
 
-<|editable_region_end|>
+
 ```