--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/StaXParser.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__SAXBuilder/StaXParser.java
@@ -1,6 +1,5 @@
-StaXParser.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package org.openmrs.module.dhisreport.api.syncmodel;
 
 import java.io.ByteArrayInputStream;
@@ -38,7 +37,8 @@
 		List<SyncDataElementGroup> items = new ArrayList<SyncDataElementGroup>();
 		try {
 			// First create a new XMLInputFactory
-			XMLInputFactory inputFactory = XMLInputFactory.newInstance();
+			org.jdom2.input.SAXBuilder inputFactory = new org.jdom2.input.SAXBuilder();
+			
 			// Setup a new eventReader
 			// InputStream in = new FileInputStream(configFile);
 			XMLEventReader eventReader = inputFactory.createXMLEventReader(in);
@@ -98,7 +98,7 @@
 		List<SyncDataElement> items = new ArrayList<SyncDataElement>();
 		try {
 			// First create a new XMLInputFactory
-			XMLInputFactory inputFactory = XMLInputFactory.newInstance();
+			org.jdom2.input.SAXBuilder inputFactory = new org.jdom2.input.SAXBuilder();
 			// Setup a new eventReader
 			InputStream stream = new ByteArrayInputStream(html
 					.getBytes("UTF-8"));
@@ -166,7 +166,7 @@
 		List<SyncDataSet> items = new ArrayList<SyncDataSet>();
 		try {
 			// First create a new XMLInputFactory
-			XMLInputFactory inputFactory = XMLInputFactory.newInstance();
+			org.jdom2.input.SAXBuilder inputFactory = new org.jdom2.input.SAXBuilder();
 			// Setup a new eventReader
 			InputStream stream = new ByteArrayInputStream(html
 					.getBytes("UTF-8"));
@@ -233,7 +233,7 @@
 		List<SyncCategoryCombo> items = new ArrayList<SyncCategoryCombo>();
 		try {
 			// First create a new XMLInputFactory
-			XMLInputFactory inputFactory = XMLInputFactory.newInstance();
+			org.jdom2.input.SAXBuilder inputFactory = new org.jdom2.input.SAXBuilder();
 			// Setup a new eventReader
 			InputStream stream = new ByteArrayInputStream(html
 					.getBytes("UTF-8"));
@@ -303,7 +303,7 @@
 		List<SyncCategoryOptionCombo> items = new ArrayList<SyncCategoryOptionCombo>();
 		try {
 			// First create a new XMLInputFactory
-			XMLInputFactory inputFactory = XMLInputFactory.newInstance();
+			org.jdom2.input.SAXBuilder inputFactory = new org.jdom2.input.SAXBuilder();
 			// Setup a new eventReader
 			InputStream stream = new ByteArrayInputStream(html
 					.getBytes("UTF-8"));
@@ -372,7 +372,7 @@
 		String periodType = new String();
 		try {
 			// First create a new XMLInputFactory
-			XMLInputFactory inputFactory = XMLInputFactory.newInstance();
+			org.jdom2.input.SAXBuilder inputFactory = new org.jdom2.input.SAXBuilder();
 			// Setup a new eventReader
 			InputStream stream = new ByteArrayInputStream(html
 					.getBytes("UTF-8"));
@@ -401,5 +401,5 @@
 		return periodType;
 	}
 }
-<|editable_region_end|>
+
 ```