--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/TagsSyncJob.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/InputFactory__TO__SAXBuilder/TagsSyncJob.java
@@ -109,7 +109,8 @@
 		}
 
 		try{
-			XMLInputFactory factory = XMLInputFactory.newInstance();
+			org.jdom2.input.SAXBuilder factory = new org.jdom2.input.SAXBuilder();
+			
 			XMLStreamReader xsr = factory.createXMLStreamReader(getTagCollectionssXmlInputStream());
 			xsr.next();
 			TransformerFactory tf = TransformerFactory.newInstance();
@@ -144,7 +145,7 @@
 		}
 
 		try{
-			XMLInputFactory factory = XMLInputFactory.newInstance();
+			org.jdom2.input.SAXBuilder factory = new org.jdom2.input.SAXBuilder();
 			XMLStreamReader xsr = factory.createXMLStreamReader(getTagsXmlInputStream());
 			xsr.next();
 			TransformerFactory tf = TransformerFactory.newInstance();
@@ -254,4 +255,4 @@
 		}
 	}
 
-}
+}