--- Bill.java
+++ Bill.java
@@ -0,0 +1,2 @@
+
+
@@ -30,0 +33 @@
+	
@@ -39 +42,4 @@
-				ResultSet rs = st.executeQuery("select * from Customer where Name='"+var.sCustomerName+"'");
+				String sql = "select * from Customer where Name= ?";
+				PreparedStatement ps = con.prepareStatement(sql);
+				ps.setString(1, var.sCustomerName);
+				ResultSet rs = ps.executeQuery();
@@ -86 +92 @@
-
+	
@@ -118 +124,2 @@
-}+}
+
