--- ff.py
+++ ff.py
@@ -0,0 +1,2 @@
+
+
@@ -39,11 +41,11 @@
-      a='Mozilla/5.0 (Linux; Android'
-      b=random.choice(['5.1.1' , '6.0.1' , '7.1.1' , '12' , '13' , '14' , '15'])
-      y=random.choice(['SM-J320H' , 'SM-J3109' , 'J320FN' , 'SM-J320P' , 'SM-J320F' , 'SM-J320G' , 'SM-J320Y'])
-      c='Build/LMY47X; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/'
-      d=random.randrange(40,115)
-      e='0'
-      f=random.randrange(3000,6000)
-      g=random.randrange(20,100)
-      h='Mobile Safari/537.36'
-      ug=(f"{a} {b}; {y} {c}{d}.{e}.{f}.{g} {h}")
-      ugen.append(ug)
+            a='Mozilla/5.0 (Linux; Android'
+            b=random.choice(['5.1.1' , '6.0.1' , '7.1.1' , '12' , '13' , '14' , '15'])
+            y=random.choice(['SM-J320H' , 'SM-J3109' , 'J320FN' , 'SM-J320P' , 'SM-J320F' , 'SM-J320G' , 'SM-J320Y'])
+            c='Build/LMY47X; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/'
+            d=random.randrange(40,115)
+            e='0'
+            f=random.randrange(3000,6000)
+            g=random.randrange(20,100)
+            h='Mobile Safari/537.36'
+            ug=(f"{a} {b}; {y} {c}{d}.{e}.{f}.{g} {h}")
+            ugen.append(ug)
@@ -205 +207,2 @@
-fuck()+fuck()
+
