--- truss.py
+++ truss.py
@@ -0,0 +1,2 @@
+
+
@@ -27,4 +29,4 @@
-    start = [5, 3, 6, 4, 4, 2, 5, 6, 3, 4]
-    finish = [3, 1, 4, 2, 3, 1, 4, 3, 2, 1]
-    phi = np.array([0, 0, 0, 0, 90, 90, -45, 45, -45, 45])*pi/180
-    L = np.array([Ls, Ls, Ls, Ls, Ls, Ls, Ld, Ld, Ld, Ld])
+    start = [5, 3, 6, 4, 4, 2, 5, 6, 3, 4, 3]
+    finish = [3, 1, 4, 2, 3, 1, 4, 3, 2, 1, 4]
+    phi = np.array([0, 0, 0, 0, 90, 90, -45, 45, -45, 45, 90])*pi/180
+    L = np.array([Ls, Ls, Ls, Ls, Ls, Ls, Ld, Ld, Ld, Ld, Ls])
@@ -57,0 +60,2 @@
+        S[i, idx] = Ssub
+        
@@ -59 +62,0 @@
-        S[i, idx] = Ssub
@@ -67,0 +71 @@
+    
@@ -75,0 +80,2 @@
+    
+        
@@ -147 +153,2 @@
-    return idx+    return idx
+
