--- NesCodeSecExamples/src/main/java/com/Scenario1/base/DocumentBuilder/ItemProcessor.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/DocumentBuilder__TO__SAXParser/ItemProcessor.java
@@ -1,6 +1,5 @@
-ItemProcessor.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package org.akaza.openclinica.controller.openrosa.processor;
 
 import java.io.StringReader;
@@ -102,8 +101,9 @@
         logger.info("Executing Item Processor.");
         ArrayList<HashMap> listOfUploadFilePaths =container.getListOfUploadFilePaths();        
 
-        DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
-        DocumentBuilder db = dbf.newDocumentBuilder();
+        javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+
+        javax.xml.parsers.SAXParser db = spf.newSAXParser();
         InputSource is = new InputSource();
         is.setCharacterStream(new StringReader(container.getRequestBody()));
         Document doc = db.parse(is);
@@ -426,5 +426,5 @@
     }
 }
 
-<|editable_region_end|>
+
 ```