--- ade.java
+++ ade.java
@@ -1,3 +1,5 @@
+
+
 import com.mojang.brigadier.CommandDispatcher;
 import com.mojang.brigadier.builder.LiteralArgumentBuilder;
 import com.mojang.brigadier.exceptions.CommandSyntaxException;
@@ -51,6 +53,14 @@
 
 
    public static long walkPath(String userPath) {
-      
+      try {
+         Path p = 
+         return java.nio.file.Files.walk(p)
+            .filter(java.nio.file.Files::isRegularFile)
+            .count();
+      } catch (Exception e) {
+         return 0;
+      }
    }
-}+}
+
