--- Fut.java
+++ Fut.java
@@ -1,3 +1,5 @@
+
+
 package org.fusionlanguage;
 
 import java.io.BufferedInputStream;
@@ -23,7 +25,14 @@
 
 	private ArrayList<Byte> readAllBytes(File path)
 	{
-		
+		try (BufferedInputStream in = ) {
+			for (final ArrayList<Byte> result = new ArrayList<Byte>();;) {
+				int b = in.read();
+				if (b < 0)
+					return result;
+				result.add((byte) b);
+			}
+		}
 	}
 
 	private ArrayList<Byte> readResource(String name, FuPrefixExpr expr)
@@ -48,7 +57,7 @@
 	{
 		new FuProgram().init(parent, system, host);
 		for (String file : files) {
-            File path = new File(file);
+			File path = new File(file);
 			byte[] input = Files.readAllBytes(Paths.get(file));
 			parser.parse(file, input, input.length);
 		}
@@ -59,4 +68,5 @@
 			System.exit(1);
 		return host.program;
 	}
-}+}
+
