--- rpn.java
+++ rpn.java
@@ -24,7 +24,15 @@
 	}
 
 	public void readFile(String filePath) throws Exception {
-		
+		Path file = 
+	   	try (InputStream in = Files.newInputStream(file);
+			BufferedReader reader = new BufferedReader(new InputStreamReader(in))) {
+			    String line = "";
+			    while ((line = reader.readLine()) != null) {
+			        equation curEquation = new equation(line);
+			        equations.add(curEquation);
+			    }
+			} 
 	}
-	
+
 }