--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/PRConfig.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXReader__TO__SAXBuilder/PRConfig.java
@@ -1,4 +1,3 @@
-
 package com.pega.gcs.logviewer.systemstate.model;
 
 import java.io.StringReader;
@@ -100,7 +99,10 @@
 
         if (xmlContent != null) {
 
-            SAXReader saxReader = new SAXReader();
+            // Replace DOM4J (SAXReader) with JDOM2 (SAXBuilder) for XML parsing
+
+            org.jdom2.input.SAXBuilder saxBuilder = new org.jdom2.input.SAXBuilder();
+
 
             StringReader stringReader = new StringReader(xmlContent);
 
@@ -125,4 +127,4 @@
         }
 
     }
-}
+}