--- SQLiteJDBCExample.java
+++ SQLiteJDBCExample.java
@@ -0,0 +1,2 @@
+
+
@@ -19,0 +22 @@
+
@@ -113 +116,4 @@
-            final PreparedStatement tkStmt = con.prepareStatement("INSERT INTO take VALUES "+"("+courseId+", "+studentId+", "+String.format("%1$tY-%1$tm-%1$td %1$tT", new Date())+")");
+            final PreparedStatement tkStmt = con.prepareStatement("INSERT INTO take VALUES (?, ?, ?)");
+            tkStmt.setLong(1, courseId);
+            tkStmt.setLong(2, studentId);
+            tkStmt.setString(3, String.format("%1$tY-%1$tm-%1$td %1$tT", new Date()));
@@ -125 +130,0 @@
-
@@ -181 +185,0 @@
-
@@ -204 +208 @@
-        final ResultSet rs = stmt.executeQuery("SELECT * FROM take WHERE course_id = " + courseId + " AND student_id = " + studentId);
+        final ResultSet rs = stmt.executeQuery("SELECT * FROM take WHERE course_id = " + courseId + " AND student_id = " + studentId)
@@ -210 +214,2 @@
-}+}
+
