--- user.java
+++ user.java
@@ -0,0 +1,2 @@
+
+
@@ -14,0 +17 @@
+import java.sql.PreparedStatement;
@@ -46,0 +50 @@
+	
@@ -75,0 +80 @@
+
@@ -114 +119,2 @@
-                    String query = "SELECT * FROM league WHERE player_id = " + pid;
+                    String query = "SELECT * FROM league WHERE player_id = ?";
+                    PreparedStatement preparedStmt;
@@ -116,3 +122,5 @@
-                    Statement stmt = Con.createStatement();
-                    res = stmt.executeQuery(query);
-                    res.next();
+                    int id  = pid;
+                    preparedStmt = Con.prepareStatement(query);
+                    preparedStmt.setInt   (1, id);
+                    res = preparedStmt.executeQuery();
+            		res.next();
@@ -138 +146 @@
-            } 
+            }
@@ -156 +164 @@
-					query = "UPDATE user Set Total = " +t;
+					query = "UPDATE user Set Total = " +t
@@ -166,0 +175 @@
+
