--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/SAXBuilderFuzzer.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXBuilder__TO__SAXParser/SAXBuilderFuzzer.java
@@ -30 +30,4 @@
-    SAXBuilder sax = new SAXBuilder();
+    javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+    
+    
+    javax.xml.parsers.SAXParser sax = spf.newSAXParser();
@@ -43 +46 @@
-}
+}