--- FileService.java
+++ FileService.java
@@ -0,0 +1,2 @@
+
+
@@ -24,5 +26 @@
-		Path filePath = Paths.get(uploadDirectory).resolve(filename).normalize();
-        
-		if(!Files.exists(filePath)) {
-			throw new IOException("El archivo no existe");
-		}
+		Path filePath = checkFileExists(filename);
@@ -35,5 +33 @@
-		Path filePath = Paths.get(uploadDirectory).resolve(filename).normalize();
-        
-		if(!Files.exists(filePath)) {
-			throw new IOException("El archivo no existe");
-		}
+		Path filePath = checkFileExists(filename);
@@ -46,0 +41,4 @@
+
+// check if file exists method
+	
+
@@ -90 +88,2 @@
-}+}
+
