--- IndexController.java
+++ IndexController.java
@@ -0,0 +1,2 @@
+
+
@@ -37,0 +40 @@
+	@RequestMapping("/index")
@@ -42,0 +46 @@
+	@RequestMapping("/admin")
@@ -60,0 +65 @@
+	@RequestMapping("/logout")
@@ -92,0 +98 @@
+	
@@ -94 +100,2 @@
-		Admin admin = loginIndexServiceImpl.getAdminByToken(token);
+		// TO DO: Implement token validation and admin retrieval logic
+		Admin admin = userser.getAdminByToken(token);
@@ -98 +105,3 @@
-}+}
+
+
