--- NesCodeSecExamples/src/main/java/com/Scenario8/base/chatbot.java
+++ NesCodeSecExamples/src/main/java/com/Scenario8/history/chatbot.java
@@ -7,7 +7,7 @@
 public class Chatbot {
 
     
-    private static final String GEMINI_API_KEY = "AIzaSyC6LV95V8bqs8BYQyYeTqkgdRTynWPrIoE";
+    private static final String GEMINI_API_KEY = os.getenv("KEY");
     private static final String GEMINI_API_URL = "https://generativelanguage.googleapis.com/v1beta/models/gemini-1.5-flash:generateContent";  
 
     public static void main(String[] args) {
