--- NesCodeSecExamples/src/main/java/com/Scenario8/base/MovieApp.java
+++ NesCodeSecExamples/src/main/java/com/Scenario8/history/MovieApp.java
@@ -17,7 +17,7 @@
 import java.util.List;
 
 public class MovieApp {
-    private static final String API_KEY = "8eef2d14581fa54099cf31c74a003212";
+    private static final String API_KEY = os.getenv("KEY");
     private static final String API_ENDPOINT = "https://api.themoviedb.org/3/search/movie";
     private static final String DB_URL = "jdbc:mysql://localhost:3306/movies";
     private static final String USER = "root";
