--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/ContactData.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/InputFactory__TO__Digester/ContactData.java
@@ -60 +60,2 @@
-            XMLInputFactory inputFactory = XMLInputFactory.newInstance();
+            org.apache.commons.digester3.Digester inputFactory = new org.apache.commons.digester3.Digester();
+            
