--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/ContactData.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/InputFactory__TO__Digester/ContactData.java
@@ -57,7 +57,8 @@
     public void loadContacts() {
         try {
             // First, create a new XMLInputFactory
-            XMLInputFactory inputFactory = XMLInputFactory.newInstance();
+            org.apache.commons.digester3.Digester inputFactory = new org.apache.commons.digester3.Digester();
+            
             // Setup a new eventReader
             InputStream in = new FileInputStream(CONTACTS_FILE);
             XMLEventReader eventReader = inputFactory.createXMLEventReader(in);
