--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/hwxml.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__Digester/hwxml.java
@@ -83,7 +83,8 @@
     {
         if( !this.newfile )
         {
-            XMLInputFactory inputFactory = XMLInputFactory.newInstance();
+            org.apache.commons.digester3.Digester inputFactory = new org.apache.commons.digester3.Digester();
+            
 			
 			InputStream in = null;
 			
@@ -173,4 +174,4 @@
         XMLEventFactory eventFactory = XMLEventFactory.newInstance();
         return eventFactory.createCharacters(value);
     }
-}
+}