--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/MetricsDefManager.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__Digester/MetricsDefManager.java
@@ -147,7 +147,9 @@
 		if (in == null)return groups;
 		
 	    XMLStreamReader reader = null;
-	    XMLInputFactory inputFactory = XMLInputFactory.newInstance();//will be used only once at startup time
+	    org.apache.commons.digester3.Digester inputFactory = new org.apache.commons.digester3.Digester();
+	    
+//will be used only once at startup time
 		try
 		{
 		  reader = inputFactory.createXMLStreamReader(new java.io.InputStreamReader(in));
@@ -334,4 +336,4 @@
 		return mg;
 	}
 
-}
+}