--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/TagsSyncJob.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__Digester/TagsSyncJob.java
@@ -1,6 +1,5 @@
-TagsSyncJob.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 /**********************************************************************************
  * $URL$
  * $Id$
@@ -112,7 +111,8 @@
 		}
 
 		try{
-			XMLInputFactory factory = XMLInputFactory.newInstance();
+			org.apache.commons.digester3.Digester factory = new org.apache.commons.digester3.Digester();
+			
 			XMLStreamReader xsr = factory.createXMLStreamReader(getTagCollectionssXmlInputStream());
 			xsr.next();
 			TransformerFactory tf = TransformerFactory.newInstance();
@@ -147,7 +147,7 @@
 		}
 
 		try{
-			XMLInputFactory factory = XMLInputFactory.newInstance();
+			org.apache.commons.digester3.Digester factory = new org.apache.commons.digester3.Digester();
 			XMLStreamReader xsr = factory.createXMLStreamReader(getTagsXmlInputStream());
 			xsr.next();
 			TransformerFactory tf = TransformerFactory.newInstance();
@@ -259,5 +259,5 @@
 
 }
 
-<|editable_region_end|>
+
 ```