--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/XxeInputFactory.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__Digester/XxeInputFactory.java
@@ -13,7 +13,8 @@
 public class XxeInputFactory {
 
     public void negative(InputStream is) throws SAXException, IOException, XMLStreamException {
-        XMLInputFactory factory = XMLInputFactory.newInstance();
+        org.apache.commons.digester3.Digester factory = new org.apache.commons.digester3.Digester();
+        
         <fim_suffix>
     }
 }
