--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/XxeInputFactory.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__Digester/XxeInputFactory.java
@@ -1,6 +1,5 @@
-XxeInputFactory.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 
 package examples;
 
@@ -15,8 +14,9 @@
 public class XxeInputFactory {
 
     public void negative(InputStream is) throws SAXException, IOException, XMLStreamException {
-        XMLInputFactory factory = XMLInputFactory.newInstance();
+        org.apache.commons.digester3.Digester factory = new org.apache.commons.digester3.Digester();
+        
     }
 }
-<|editable_region_end|>
+
 ```