--- ImageSample.java
+++ ImageSample.java
@@ -0,0 +1,2 @@
+
+
@@ -18,3 +20,3 @@
-    private static String accessKeyId = "<accessKeyId>";
-    private static String accessKeySecret = "<accessKeySecret>";
-    private static String bucketName = "<bucketName>";
+    private static String accessKeyId = "AKIAIOSFODNN7EXAMPLE";
+    private static String accessKeySecret = 
+    private static String bucketName = "my-storage-bucket";
@@ -96 +98,2 @@
-}+}
+
