--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/StaxParser.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__DocumentBuilder/StaxParser.java
@@ -30,7 +30,9 @@
 
     @Override
     protected void parseInputStream(Reader reader) {
-        XMLInputFactory factory = XMLInputFactory.newInstance();
+        javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+        
+        javax.xml.parsers.DocumentBuilder factory = dbf.newDocumentBuilder();
         try {
             XMLStreamReader streamReader = factory.createXMLStreamReader(reader);
             processReader(streamReader);
@@ -43,7 +45,8 @@
 
     @Override
     protected void parseInputStream(InputStream inputStream) {
-        XMLInputFactory factory = XMLInputFactory.newInstance();
+        javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+        javax.xml.parsers.DocumentBuilder factory = dbf.newDocumentBuilder();
         try {
             XMLStreamReader streamReader = factory.createXMLStreamReader(inputStream);
             processReader(streamReader);
@@ -208,4 +211,4 @@
     }
 
 
-}
+}