--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/TagsSyncJob.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/InputFactory__TO__DocumentBuilder/TagsSyncJob.java
@@ -112 +112,3 @@
-			XMLInputFactory factory = XMLInputFactory.newInstance();
+			javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+			
+			javax.xml.parsers.DocumentBuilder factory = dbf.newDocumentBuilder();
@@ -147 +149,2 @@
-			XMLInputFactory factory = XMLInputFactory.newInstance();
+			javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+			javax.xml.parsers.DocumentBuilder factory = dbf.newDocumentBuilder();
@@ -257 +260 @@
-}
+}