--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/XxeInputFactory.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__DocumentBuilder/XxeInputFactory.java
@@ -13,7 +13,9 @@
 public class XxeInputFactory {
 
     public void negative(InputStream is) throws SAXException, IOException, XMLStreamException {
-        XMLInputFactory factory = XMLInputFactory.newInstance();
+        javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+        
+        javax.xml.parsers.DocumentBuilder factory = dbf.newDocumentBuilder();
         <fim_suffix>
     }
 }
