--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/MessageUtil.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXReader__TO__SAXParser/MessageUtil.java
@@ -1,6 +1,5 @@
-MessageUtil.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package com.qihua.ofbiz.weixin.util;
 
 /**
@@ -114,7 +113,9 @@
         // 从request中取得输入流
         InputStream inputStream = request.getInputStream();
         // 读取输入流
-        SAXReader reader = new SAXReader();
+        javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+        
+        javax.xml.parsers.SAXParser reader = spf.newSAXParser();
         Document document = reader.read(inputStream);
         // 得到xml根元素
         Element root = document.getRootElement();
@@ -197,5 +198,5 @@
     });
 }
 
-<|editable_region_end|>
+
 ```