--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/MessageUtil.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXReader__TO__SAXParser/MessageUtil.java
@@ -114 +114,3 @@
-        SAXReader reader = new SAXReader();
+        javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+        
+        javax.xml.parsers.SAXParser reader = spf.newSAXParser();
@@ -195 +197 @@
-}
+}