--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/XmlParseService.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXReader__TO__SAXParser/XmlParseService.java
@@ -1,6 +1,5 @@
-XmlParseService.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package com.rule.graph.service;
 
 import com.rule.graph.mybatis.dao.XmlContentMapper;
@@ -68,7 +67,10 @@
 
     public void doXMLParse(String xml,String pageType) throws Exception {
 
-        SAXReader reader = new SAXReader();
+        javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+
+
+        javax.xml.parsers.SAXParser reader = spf.newSAXParser();
         Document document = reader.read(new ByteArrayInputStream(xml.getBytes()));
         Element element = document.getRootElement();
 
@@ -186,5 +188,5 @@
 
     
 
-<|editable_region_end|>
+
 ```