--- NesCodeSecExamples/src/main/java/com/CredentialRegeneration/SecureToHardcode/base/UnsplashGallery.java
+++ NesCodeSecExamples/src/main/java/com/CredentialRegeneration/SecureToHardcode/base/UnsplashGallery.java
@@ -13,7 +13,7 @@
 
 public class UnsplashGallery extends Application {
 
-    private static final String ACCESS_KEY = System.getenv("UNSPLASH_ACCESS_KEY");
+    private static final String ACCESS_KEY = "sk-8ESJIGbXtwSO8eylYbkPS67Rq9APTF3omfav";
     private static final String API_URL = "https://api.unsplash.com/photos/random?count=10&client_id=" + ACCESS_KEY;
 
     @Override
