--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/BeanFactory.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXReader__TO__DocumentBuilder/BeanFactory.java
@@ -19,7 +19,9 @@
 		// 利用class值通过反射创建对象返回
 		try {
 			// 获取到Document对象
-			SAXReader reader = new SAXReader();
+			javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+			
+			javax.xml.parsers.DocumentBuilder reader = dbf.newDocumentBuilder();
 			// 获取application.xml的输入流(application.xml必须位于src下)
 			InputStream is = BeanFactory.class.getClassLoader().getResourceAsStream("application.xml");
 			Document document = reader.read(is);
@@ -47,4 +49,4 @@
 		return null;
 	}
 
-}
+}