--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/DocumentUtilities.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXReader__TO__DocumentBuilder/DocumentUtilities.java
@@ -49,7 +49,9 @@
 
   public static Document read(String source) throws AnathemaException {
     try {
-      SAXReader saxReader = new SAXReader();
+      javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+      
+      javax.xml.parsers.DocumentBuilder saxReader = dbf.newDocumentBuilder();
       return saxReader.read(new StringReader(source));
     } catch (DocumentException exception) {
       throw new AnathemaException(exception);
@@ -66,7 +68,8 @@
 
   public static Document read(InputStream in) throws PersistenceException {
     try {
-      SAXReader saxReader = new SAXReader();
+      javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+      javax.xml.parsers.DocumentBuilder saxReader = dbf.newDocumentBuilder();
       return saxReader.read(in, null);
     } catch (DocumentException exception) {
       throw new PersistenceException(exception);
