--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/CitygmlPackFile.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXParser__TO__Digester/CitygmlPackFile.java
@@ -31,2 +31 @@
-        SAXParserFactory factory = SAXParserFactory.newInstance();
-        factory.setValidating(false);
+        org.apache.commons.digester3.Digester parser = new org.apache.commons.digester3.Digester();
@@ -34 +32,0 @@
-        SAXParser parser = factory.newSAXParser();
@@ -39 +37 @@
-}
+}