--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/PhysicsEditorShapeLibrary.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXParser__TO__Digester/PhysicsEditorShapeLibrary.java
@@ -1,6 +1,5 @@
-PhysicsEditorShapeLibrary.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package se.treplex.sketchytruck;
 
 /**
@@ -106,7 +105,8 @@
     }
 
     private void append(Context context, String name, float pixelToMeterRatio) {
-        SAXParserFactory factory = SAXParserFactory.newInstance();
+        org.apache.commons.digester3.Digester factory = new org.apache.commons.digester3.Digester();
+        
         try {
             SAXParser parser = factory.newSAXParser();
             ShapeLoader handler = new ShapeLoader(shapes, pixelToMeterRatio);
@@ -233,5 +233,5 @@
 }
 
 
-<|editable_region_end|>
+
 ```