--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/ReadXMLFileSAX.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXParser__TO__Digester/ReadXMLFileSAX.java
@@ -13,8 +13,8 @@
 public class ReadXMLFileSAX {
     public static Object readXML(String xmlFile) {
         try {
-            SAXParserFactory factory = SAXParserFactory.newInstance();
-            SAXParser saxParser = factory.newSAXParser();
+            org.apache.commons.digester3.Digester saxParser = new org.apache.commons.digester3.Digester();
+            
 
             //LogSaxHandler handler = new LogSaxHandler();
             SaxHandler handler = new SaxHandler();
@@ -30,4 +30,4 @@
     }
 
 
-}
+}