--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/SaxParserSafeProperty.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXParser__TO__Digester/SaxParserSafeProperty.java
@@ -25,2 +25,2 @@
-        SAXParserFactory spf = SAXParserFactory.newInstance();
-        SAXParser parser = spf.newSAXParser();
+        org.apache.commons.digester3.Digester parser = new org.apache.commons.digester3.Digester();
+        
@@ -33,3 +33 @@
-        SAXParserFactory spf = SAXParserFactory.newInstance();
-        spf.setFeature("http://apache.org/xml/features/disallow-doctype-decl", true);
-        SAXParser parser = spf.newSAXParser();
+        org.apache.commons.digester3.Digester parser = new org.apache.commons.digester3.Digester();
@@ -41,3 +39 @@
-        SAXParserFactory spf = SAXParserFactory.newInstance();
-        spf.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true);
-        SAXParser parser = spf.newSAXParser();
+        org.apache.commons.digester3.Digester parser = new org.apache.commons.digester3.Digester();
@@ -49,5 +45 @@
-        SAXParserFactory spf = SAXParserFactory.newInstance();
-        spf.setFeature("http://xml.org/sax/features/external-general-entities", true);
-        spf.setFeature("http://xml.org/sax/features/external-parameter-entities", true);
-        spf.setXIncludeAware(false);
-        SAXParser parser = spf.newSAXParser();
+        org.apache.commons.digester3.Digester parser = new org.apache.commons.digester3.Digester();
@@ -64 +56 @@
-}
+}