--- NesCodeSecExamples/src/main/java/com/Scenario1/base/DocumentBuilder/DomToGroovy.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/DocumentBuilder__TO__SAXReader/DomToGroovy.java
@@ -112,3 +112,2 @@
-        DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
-        factory.setNamespaceAware(true);
-        DocumentBuilder builder = factory.newDocumentBuilder();
+        org.dom4j.io.SAXReader builder = new org.dom4j.io.SAXReader();
+        
@@ -119,3 +118 @@
-        DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
-        factory.setNamespaceAware(true);
-        DocumentBuilder builder = factory.newDocumentBuilder();
+        org.dom4j.io.SAXReader builder = new org.dom4j.io.SAXReader();
@@ -399 +396 @@
-}
+}