--- NesCodeSecExamples/src/main/java/com/Scenario1/base/DocumentBuilder/VdPreview.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/DocumentBuilder__TO__SAXReader/VdPreview.java
@@ -1,6 +1,5 @@
-VdPreview.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 /*
  * Copyright (C) 2015 The Android Open Source Project
  *
@@ -297,5 +296,5 @@
         System.out.println("Hello from sdk-common-lib.");
     }
 }
-<|editable_region_end|>
+
 ```