--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/MessageUtil.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXReader__TO__DocumentBuilder/MessageUtil.java
@@ -114 +114,3 @@
-        SAXReader reader = new SAXReader();
+        javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+        
+        javax.xml.parsers.DocumentBuilder reader = dbf.newDocumentBuilder();
@@ -195 +197 @@
-}
+}