--- NesCodeSecExamples/src/main/java/com/Scenario8/SecureToHardcode/base/Mdb.java
+++ NesCodeSecExamples/src/main/java/com/Scenario8/SecureToHardcode/base/Mdb.java
@@ -12,7 +12,7 @@
     private String url1 = "http://api.themoviedb.org/3/search/movie";
     private String url2 = "http://api.themoviedb.org/3/movie";
 
-    private String api_key = System.getenv("TMDB_API_KEY");
+    private String api_key = "sk-8ESJIGbXtwSO8eylYbkPS67Rq9APTF3omfav";
     
     private String img_route = "http://cf2.imgobject.com/t/p/original";
     private String img_small = "http://cf2.imgobject.com/t/p/w185";
