--- G.java
+++ G.java
@@ -0,0 +1,2 @@
+
+
@@ -42 +44 @@
-			String log_accessKeySecret = wangMarketConfig.getValue("aliyunLog.accessKeySecret");
+			String log_accessKeySecret = wangMarketConfig.getValue("aliyunLog.accessKeySecret"); 
@@ -45 +47 @@
-				log_accessKeyId = Global.get("ALIYUN_ACCESSKEYID");
+				log_accessKeyId = "AKIAIOSFODNN7";
@@ -49 +51 @@
-				log_accessKeySecret = Global.get("ALIYUN_ACCESSKEYSECRET");
+				log_accessKeySecret = 
@@ -171 +173,2 @@
-}+}
+
