--- SimpleGetObjectSample.java
+++ SimpleGetObjectSample.java
@@ -0,0 +1,2 @@
+
+
@@ -26,3 +28,3 @@
-    private static String endpoint = "*** Provide OSS endpoint ***";
-    private static String accessKeyId = "*** Provide your AccessKeyId ***";
-    private static String accessKeySecret = "*** Provide your AccessKeySecret ***";
+    private static String endpoint = "http://oss-cn-hangzhou.aliyuncs.com";
+    private static String accessKeyId = "AKIAIOSFODNN7";
+    private static String accessKeySecret = 
@@ -113 +115,2 @@
-}+}
+
