--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/DefaultSAXPrologProcessRegistry.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXParser__TO__DocumentBuilder/DefaultSAXPrologProcessRegistry.java
@@ -42,7 +42,9 @@
 		//Element cpElement;
 
 		try {
-			SAXParser parser = SAXParserFactory.newInstance().newSAXParser();
+			javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+			
+			javax.xml.parsers.DocumentBuilder parser = dbf.newDocumentBuilder();
 			//parser.setProperty("http://xml.org/sax/features/namespaces", new Boolean(true));
 			//boolean b = parser.isNamespaceAware();
 			parser.parse(new InputSource(reader),new RegistryHandler());
@@ -141,6 +143,4 @@
 
 	
 
-}
-
-
+}