--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/LabsdbService.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXParser__TO__DocumentBuilder/LabsdbService.java
@@ -199,7 +199,9 @@
          }
        }
       //generatorWiremapData(servers,pduIDListMap,networkIDListMap,labsdbClient);
-      SAXParserFactory spf = SAXParserFactory.newInstance();
+      javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+      
+      javax.xml.parsers.DocumentBuilder spf = dbf.newDocumentBuilder();
       WiremapSaxHandler handler = new WiremapSaxHandler(wirmMap_node);
       SAXParser parser = null;
       try {
@@ -335,4 +337,4 @@
       return assetNameAndIdMap;
    }
 
-}
+}