--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/PhysicsEditorShapeLibrary.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXParser__TO__DocumentBuilder/PhysicsEditorShapeLibrary.java
@@ -103,7 +103,9 @@
     }
 
     private void append(Context context, String name, float pixelToMeterRatio) {
-        SAXParserFactory factory = SAXParserFactory.newInstance();
+        javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+        
+        javax.xml.parsers.DocumentBuilder factory = dbf.newDocumentBuilder();
         try {
             SAXParser parser = factory.newSAXParser();
             ShapeLoader handler = new ShapeLoader(shapes, pixelToMeterRatio);
@@ -227,5 +229,4 @@
             vertices = vectorList.toArray(new Vector2[vectorList.size()]);
         }
     }
-}
-
+}