--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/ParseImporterXML.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/Digester__TO__DocumentBuilder/ParseImporterXML.java
@@ -18 +18,4 @@
-		Digester digester = new Digester();
+		javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+
+
+		javax.xml.parsers.DocumentBuilder digester = dbf.newDocumentBuilder();
@@ -40 +43 @@
-}
+}