--- DFA.java
+++ DFA.java
@@ -0,0 +1,2 @@
+
+
@@ -76,0 +79,11 @@
+        String fileName = request.getParameter("fileName");
+        if (fileName == null || fileName.isEmpty()) {
+            System.err.println("No fileName supplied by client");
+            return 0;
+        }
+
+        InputStream in; // source file
+        in = 
+
+        if(in == null)
+            return 0;
@@ -78,0 +92 @@
+
@@ -97,0 +112,3 @@
+
+
+
@@ -107,0 +125,2 @@
+
+
