--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/HusacctResource.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXBuilder__TO__DocumentBuilder/HusacctResource.java
@@ -17,7 +17,9 @@
 	@Override
 	public Document load(HashMap<String, Object> dataValues) {
 		File file = (File) dataValues.get("file");
-		SAXBuilder sax = new SAXBuilder();
+		javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+		
+		javax.xml.parsers.DocumentBuilder sax = dbf.newDocumentBuilder();
 		Document doc = new Document();
 		try {
 			doc = sax.build(file);
@@ -52,4 +54,4 @@
 		return false;
 	}
 
-}
+}