--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/JDomParser.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXBuilder__TO__DocumentBuilder/JDomParser.java
@@ -22,7 +22,9 @@
 
     public List<Element> getAllTitles() {
         try {
-            SAXBuilder builder = new SAXBuilder();
+            javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+            
+            javax.xml.parsers.DocumentBuilder builder = dbf.newDocumentBuilder();
             Document doc = builder.build(this.getFile());
             Element tutorials = doc.getRootElement();
             List<Element> titles = tutorials.getChildren("tutorial");
@@ -36,7 +38,8 @@
 
     public Element getNodeById(String id) {
         try {
-            SAXBuilder builder = new SAXBuilder();
+            javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+            javax.xml.parsers.DocumentBuilder builder = dbf.newDocumentBuilder();
             Document document = (Document) builder.build(file);
             String filter = "//*[@tutId='" + id + "']";
             XPathFactory xFactory = XPathFactory.instance();
@@ -58,4 +61,4 @@
         this.file = file;
     }
 
-}
+}