--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/SAXBuilderFuzzer.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXBuilder__TO__DocumentBuilder/SAXBuilderFuzzer.java
@@ -27,7 +27,10 @@
 public class SAXBuilderFuzzer {
   public static void fuzzerTestOneInput(FuzzedDataProvider data) {
     
-    SAXBuilder sax = new SAXBuilder();
+    javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+    
+    
+    javax.xml.parsers.DocumentBuilder sax = dbf.newDocumentBuilder();
     sax.setValidation(data.consumeBoolean());
     sax.setIgnoringElementContentWhitespace(data.consumeBoolean());
     sax.setIgnoringBoundaryWhitespace(data.consumeBoolean());
@@ -40,4 +43,4 @@
       return;
     }
   }
-}
+}