--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/Util.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXBuilder__TO__DocumentBuilder/Util.java
@@ -62,7 +62,9 @@
 
     public static Element loadRootElement(File configFile) throws CruiseControlException {
         try {
-            SAXBuilder builder = new SAXBuilder();
+            javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+            
+            javax.xml.parsers.DocumentBuilder builder = dbf.newDocumentBuilder();
             builder.setFeature("http://apache.org/xml/features/xinclude", true);
             return builder.build(configFile).getRootElement();
         } catch (Exception e) {
@@ -76,7 +78,8 @@
 
     public static Element loadRootElement(InputStream in) throws CruiseControlException {
         try {
-            SAXBuilder builder = new SAXBuilder();
+            javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+            javax.xml.parsers.DocumentBuilder builder = dbf.newDocumentBuilder();
             return builder.build(in).getRootElement();
         } catch (Exception e) {
             throw new CruiseControlException("failed to parse configuration", e);
@@ -301,4 +304,4 @@
         }
         return string;
     }
-}
+}