--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/HusacctResource.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXBuilder__TO__SAXParser/HusacctResource.java
@@ -1,6 +1,5 @@
-HusacctResource.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package husacct.control.task.resources;
 
 import java.io.File;
@@ -20,7 +19,9 @@
 	@Override
 	public Document load(HashMap<String, Object> dataValues) {
 		File file = (File) dataValues.get("file");
-		SAXBuilder sax = new SAXBuilder();
+		javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+		
+		javax.xml.parsers.SAXParser sax = spf.newSAXParser();
 		Document doc = new Document();
 		try {
 			doc = sax.build(file);
@@ -57,5 +58,5 @@
 
 }
 
-<|editable_region_end|>
+
 ```