--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/PluginTestUsername.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXBuilder__TO__SAXParser/PluginTestUsername.java
@@ -1,6 +1,5 @@
-PluginTestUsername.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package registration.plugintest;
 
 import static org.junit.Assert.*;
@@ -50,7 +49,10 @@
 	@Before
 	public void setUp() throws Exception {
 
-		SAXBuilder builder = new SAXBuilder();
+		javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+
+
+		javax.xml.parsers.SAXParser builder = spf.newSAXParser();
 		try {
 			document = builder.build(new File("./testData.xml")
 					.getCanonicalPath());
@@ -198,5 +200,5 @@
 
 }
 
-<|editable_region_end|>
+
 ```