--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/Util.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXBuilder__TO__SAXParser/Util.java
@@ -62,7 +62,10 @@
 
     public static Element loadRootElement(File configFile) throws CruiseControlException {
         try {
-            SAXBuilder builder = new SAXBuilder();
+            // Replace JDOM2 (SAXBuilder) with SAX (SAXParserFactory) for XML parsing
+            javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+            
+            javax.xml.parsers.SAXParser spf = spf.newSAXParser();
             return builder.build(configFile).getRootElement();
         } catch (Exception e) {
             throw new CruiseControlException(
@@ -75,7 +78,9 @@
 
     public static Element loadRootElement(InputStream in) throws CruiseControlException {
         try {
-            SAXBuilder builder = new SAXBuilder();
+            // Replace JDOM2 (SAXBuilder) with SAX (SAXParserFactory) for XML parsing
+            javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+            javax.xml.parsers.SAXParser spf = spf.newSAXParser();
             return builder.build(in).getRootElement();
         } catch (Exception e) {
             throw new CruiseControlException("failed to parse configuration", e);
@@ -300,4 +305,4 @@
         }
         return string;
     }
-}
+}