--- hashPassword.java
+++ hashPassword.java
@@ -1 +1,2 @@
-import java.security.MessageDigest;
+
+
@@ -8 +9,12 @@
-        
+        try {
+            // encryp password
+            String encryptionAlgorithm = 
+            
+        }
+        catch (NoSuchAlgorithmException e) {
+            e.printStackTrace();
+        }
+
+        System.out.println("Plain-text password: " + password);
+        System.out.println("Encrypted password using " + encryptionAlgorithm + ": " + encryptedpassword);
+        System.out.println(encryptedpassword.length());
@@ -10 +22,2 @@
-}+}
+
