--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/Util.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXBuilder__TO__InputFactory/Util.java
@@ -65 +65,2 @@
-            SAXBuilder builder = new SAXBuilder();
+            javax.xml.stream.XMLInputFactory builder = javax.xml.stream.XMLInputFactory.newFactory();
+            
@@ -79 +80 @@
-            SAXBuilder builder = new SAXBuilder();
+            javax.xml.stream.XMLInputFactory builder = javax.xml.stream.XMLInputFactory.newFactory();
@@ -304 +305 @@
-}
+}