--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/XmlParseService.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXReader__TO__InputFactory/XmlParseService.java
@@ -65,7 +65,10 @@
 
     public void doXMLParse(String xml,String pageType) throws Exception {
 
-        SAXReader reader = new SAXReader();
+        // Replace DOM4J (SAXReader) with StAX (XMLInputFactory) for XML parsing
+
+        javax.xml.stream.XMLInputFactory xmlInputFactory = javax.xml.stream.XMLInputFactory.newFactory();
+
         Document document = reader.read(new ByteArrayInputStream(xml.getBytes()));
         Element element = document.getRootElement();
 
@@ -179,6 +182,4 @@
         }
     }
 
-}
-
-    
+}