--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/DocumentUtilities.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXReader__TO__Digester/DocumentUtilities.java
@@ -1,6 +1,5 @@
-DocumentUtilities.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package net.sf.anathema.lib.xml;
 
 import net.sf.anathema.lib.exception.AnathemaException;
@@ -52,7 +51,8 @@
 
   public static Document read(String source) throws AnathemaException {
     try {
-      SAXReader saxReader = new SAXReader();
+      org.apache.commons.digester3.Digester saxReader = new org.apache.commons.digester3.Digester();
+      
       return saxReader.read(new StringReader(source));
     } catch (DocumentException exception) {
       throw new AnathemaException(exception);
@@ -69,7 +69,7 @@
 
   public static Document read(InputStream in) throws PersistenceException {
     try {
-      SAXReader saxReader = new SAXReader();
+      org.apache.commons.digester3.Digester saxReader = new org.apache.commons.digester3.Digester();
       return saxReader.read(in, null);
     } catch (DocumentException exception) {
       throw new PersistenceException(exception);
@@ -93,5 +93,5 @@
     return (Element) result;
   }
 }
-<|editable_region_end|>
+
 ```