--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/DocumentUtilities.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXReader__TO__Digester/DocumentUtilities.java
@@ -49,7 +49,8 @@
 
   public static Document read(String source) throws AnathemaException {
     try {
-      SAXReader saxReader = new SAXReader();
+      org.apache.commons.digester3.Digester saxReader = new org.apache.commons.digester3.Digester();
+      
       return saxReader.read(new StringReader(source));
     } catch (DocumentException exception) {
       throw new AnathemaException(exception);
@@ -66,7 +67,7 @@
 
   public static Document read(InputStream in) throws PersistenceException {
     try {
-      SAXReader saxReader = new SAXReader();
+      org.apache.commons.digester3.Digester saxReader = new org.apache.commons.digester3.Digester();
       return saxReader.read(in, null);
     } catch (DocumentException exception) {
       throw new PersistenceException(exception);
