--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/ErrorFilter.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXReader__TO__Digester/ErrorFilter.java
@@ -104,7 +104,8 @@
 
         // JAXB will expect a namespace if we try to unmarshall, but some error responses don't include
         // a namespace. In lieu of writing a SAXFilter to apply a default namespace in-line, this works just as well.
-        SAXReader saxReader = new SAXReader();
+        org.apache.commons.digester3.Digester saxReader = new org.apache.commons.digester3.Digester();
+        
 
         Document d;
         try {
@@ -133,4 +134,4 @@
         log.debug("Error: {}, message: {}, requestId: {}", new Object[]{code, message, requestId});
         return new S3Exception(message, statusCode, code, requestId);
     }
-}
+}