--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/LineMaker.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXReader__TO__Digester/LineMaker.java
@@ -1,6 +1,5 @@
-LineMaker.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package com.thinkdifferent.convertpreview.utils;
 
 import cn.hutool.core.io.FileUtil;
@@ -74,7 +73,8 @@
             strRoot = strRoot.replaceAll("\\\\", "/");
             String strPath = strRoot + "/conf/";
 
-            SAXReader reader = new SAXReader();
+            org.apache.commons.digester3.Digester reader = new org.apache.commons.digester3.Digester();
+
             Document document = reader.read(new File(strPath + "CustomLine.xml"));
             Element elmRoot = document.getRootElement();
             Element elmField = elmRoot.element(strName);
@@ -269,5 +269,5 @@
 
 }
 
-<|editable_region_end|>
+
 ```