--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/XmlParseService.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXReader__TO__Digester/XmlParseService.java
@@ -65,7 +65,8 @@
 
     public void doXMLParse(String xml,String pageType) throws Exception {
 
-        SAXReader reader = new SAXReader();
+        org.apache.commons.digester3.Digester reader = new org.apache.commons.digester3.Digester();
+
         Document document = reader.read(new ByteArrayInputStream(xml.getBytes()));
         Element element = document.getRootElement();
 
@@ -179,6 +180,4 @@
         }
     }
 
-}
-
-    
+}