--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/XMLTextConceptDigester.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/Digester__TO__SAXReader/XMLTextConceptDigester.java
@@ -1,6 +1,5 @@
-XMLTextConceptDigester.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 /**
  *  Licensed to the Apache Software Foundation (ASF) under one
  *  or more contributor license agreements.  See the NOTICE file
@@ -37,7 +36,8 @@
   public Results parseAlchemyXML(InputStream xmlReader) throws IOException, SAXException,
           URISyntaxException {
 
-    Digester digester = new Digester();
+    org.dom4j.io.SAXReader digester = new org.dom4j.io.SAXReader();
+
     digester.setValidating(false);
 
     digester.addObjectCreate("results", ConceptResults.class);
@@ -69,5 +69,5 @@
   }
 }
 
-<|editable_region_end|>
+
 ```