--- NesCodeSecExamples/src/main/java/com/CredentialRegeneration/HardcodeToSecure/base/ChatGPT.java
+++ NesCodeSecExamples/src/main/java/com/CredentialRegeneration/HardcodeToSecure/base/ChatGPT.java
@@ -10,7 +10,7 @@
 
 class ChatGPT {
     private static final String API_ENDPOINT = "https://api.openai.com/v1/completions";
-    private static final String API_KEY = 
+    private static final String API_KEY = "sk-8ESJIGbXtwSO8eylYbkPS67Rq9APTF3omfav";
     private static final String MODEL = "text-davinci-003";
     
     private String question;

--- NesCodeSecExamples/src/main/java/com/CredentialRegeneration/HardcodeToSecure/base/ChatGPT.java
+++ NesCodeSecExamples/src/main/java/com/CredentialRegeneration/HardcodeToSecure/base/ChatGPT.java
@@ -10,7 +10,7 @@
 
 class ChatGPT {
     private static final String API_ENDPOINT = "https://api.openai.com/v1/completions";
-    private static final String API_KEY = "sk-8ESJIGbXtwSO8eylYbkPS67Rq9APTF3omfav";
+    private static final String API_KEY = System.getenv("KEY");
     private static final String MODEL = "text-davinci-003";
     
     private String question;
