--- NesCodeSecExamples/src/main/java/com/CredentialRegeneration/HardcodeToSecure/base/MovieApp.java
+++ NesCodeSecExamples/src/main/java/com/CredentialRegeneration/HardcodeToSecure/base/MovieApp.java
@@ -17,7 +17,7 @@
 import java.util.List;
 
 public class MovieApp {
-    private static final String API_KEY = 
+    private static final String API_KEY = "sk-8ESJIGbXtwSO8eylYbkPS67Rq9APTF3omfav";
     private static final String API_ENDPOINT = "https://api.themoviedb.org/3/search/movie";
     private static final String DB_URL = "jdbc:mysql://localhost:3306/movies";
     private static final String USER = "root";

--- NesCodeSecExamples/src/main/java/com/CredentialRegeneration/HardcodeToSecure/base/MovieApp.java
+++ NesCodeSecExamples/src/main/java/com/CredentialRegeneration/HardcodeToSecure/base/MovieApp.java
@@ -17,7 +17,7 @@
 import java.util.List;
 
 public class MovieApp {
-    private static final String API_KEY = "sk-8ESJIGbXtwSO8eylYbkPS67Rq9APTF3omfav";
+    private static final String API_KEY = System.getenv("KEY");
     private static final String API_ENDPOINT = "https://api.themoviedb.org/3/search/movie";
     private static final String DB_URL = "jdbc:mysql://localhost:3306/movies";
     private static final String USER = "root";
