--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/DefaultSAXPrologProcessRegistry.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXParser__TO__InputFactory/DefaultSAXPrologProcessRegistry.java
@@ -42,7 +42,8 @@
 		//Element cpElement;
 
 		try {
-			SAXParser parser = SAXParserFactory.newInstance().newSAXParser();
+			javax.xml.stream.XMLInputFactory parser = javax.xml.stream.XMLInputFactory.newFactory();
+			
 			//parser.setProperty("http://xml.org/sax/features/namespaces", new Boolean(true));
 			//boolean b = parser.isNamespaceAware();
 			parser.parse(new InputSource(reader),new RegistryHandler());
@@ -141,6 +142,4 @@
 
 	
 
-}
-
-
+}