--- NesCodeSecExamples/src/main/java/com/Scenario8/base/BucketCnameSample.java
+++ NesCodeSecExamples/src/main/java/com/Scenario8/input_event/AccessPointSample/BucketCnameSample.java
@@ -10,7 +10,7 @@
 
     private static String endpoint = "https://mybucket.oss-cn-shanghai.aliyuncs.com";
     private static String accessKeyId = "LTAI4EXAMPLE123";
-    private static String accessKeySecret = "iqCHpQ4r2HqfOnl4hOeK8WDIrhUknvd";
+    private static String accessKeySecret = 
     private static String bucketName = "*** Provide bucket name ***";
 
     public static void main(String[] args) {
