--- NesCodeSecExamples/src/main/java/com/Scenario8/base/ChatGPT.java
+++ NesCodeSecExamples/src/main/java/com/Scenario8/input_event/AccessPointSample/ChatGPT.java
@@ -10,7 +10,7 @@
 
 class ChatGPT {
     private static final String API_ENDPOINT = "https://api.openai.com/v1/completions";
-    private static final String API_KEY = "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";
+    private static final String API_KEY = 
     private static final String MODEL = "text-davinci-003";
     
     private String question;
