--- Persist.java
+++ Persist.java
@@ -0,0 +1,2 @@
+
+
@@ -59 +61,2 @@
-            
+            FileInputStream streamIn = new FileInputStream(file);
+            ObjectInputStream objectinputstream = 
@@ -72,0 +76 @@
+
@@ -82,0 +87,8 @@
+            ObjectOutputStream oos = new ObjectOutputStream(fos);
+            oos.writeObject(this);
+            oos.close();
+        }
+        catch(Exception e)
+        {
+            e.printStackTrace();
+        }
@@ -84 +96,2 @@
-}+}
+
