--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/TagsSyncJob.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/InputFactory__TO__Digester/TagsSyncJob.java
@@ -112 +112,2 @@
-			XMLInputFactory factory = XMLInputFactory.newInstance();
+			org.apache.commons.digester3.Digester factory = new org.apache.commons.digester3.Digester();
+			
@@ -147 +148 @@
-			XMLInputFactory factory = XMLInputFactory.newInstance();
+			org.apache.commons.digester3.Digester factory = new org.apache.commons.digester3.Digester();
@@ -257 +258 @@
-}
+}