--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/PRConfig.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXReader__TO__SAXBuilder/PRConfig.java
@@ -1,6 +1,5 @@
-PRConfig.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 
 package com.pega.gcs.logviewer.systemstate.model;
 
@@ -103,7 +102,8 @@
 
         if (xmlContent != null) {
 
-            SAXReader saxReader = new SAXReader();
+            org.jdom2.input.SAXBuilder saxReader = new org.jdom2.input.SAXBuilder();
+
 
             StringReader stringReader = new StringReader(xmlContent);
 
@@ -130,5 +130,5 @@
     }
 }
 
-<|editable_region_end|>
+
 ```