--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/XmlParseService.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXReader__TO__SAXBuilder/XmlParseService.java
@@ -1,6 +1,5 @@
-XmlParseService.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package com.rule.graph.service;
 
 import com.rule.graph.mybatis.dao.XmlContentMapper;
@@ -68,7 +67,8 @@
 
     public void doXMLParse(String xml,String pageType) throws Exception {
 
-        SAXReader reader = new SAXReader();
+        org.jdom2.input.SAXBuilder reader = new org.jdom2.input.SAXBuilder();
+
         Document document = reader.read(new ByteArrayInputStream(xml.getBytes()));
         Element element = document.getRootElement();
 
@@ -186,5 +186,5 @@
 
     
 
-<|editable_region_end|>
+
 ```