--- NesCodeSecExamples/src/main/java/com/CredentialRegeneration/HardcodeToSecure/base/BucketArchiveDirectReadSample.java
+++ NesCodeSecExamples/src/main/java/com/CredentialRegeneration/HardcodeToSecure/base/BucketArchiveDirectReadSample.java
@@ -7,7 +7,7 @@
 public class BucketArchiveDirectReadSample {
     private static String endpoint = "https://mybucket.oss-cn-shanghai.aliyuncs.com";
     private static String accessKeyId = "LTAI4EXAMPLE123";
-    private static String accessKeySecret = "sk-8ESJIGbXtwSO8eylYbkPS67Rq9APTF3omfav";
+    private static String accessKeySecret = "sk-8ESJIGbXtwSO8eylYbkPS67Rq9APTF3omfav"
     private static String bucketName = "*** Provide bucket name ***";
     public static void main(String[] args) {
         OSS ossClient = new OSSClientBuilder().build(endpoint, accessKeyId, accessKeySecret);
