--- SqlOracle.java
+++ SqlOracle_response_1.java
@@ -1,7 +1,5 @@
-
-
 package com.taotao.cloud.workflow.biz.common.database.sql.impl;
 
 import com.taotao.cloud.workflow.biz.common.database.source.DbBase;
 import com.taotao.cloud.workflow.biz.common.database.sql.SqlBase;
 import java.sql.Connection;
@@ -124,9 +122,10 @@
         return InsertSql.batch(iInsertSqlDTO);
     }
 
     @Override
     public PreparedStatementDTO getDeleteSqlPSD(Connection conn, String deleteTable) {
-    
+        String sql = "DELETE FROM " + deleteTable;
+        PreparedStatementDTO psd = new PreparedStatementDTO(sql, conn);
+        return psd;
     }
 }
-
