--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/ErrorFilter.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXReader__TO__DocumentBuilder/ErrorFilter.java
@@ -104,7 +104,10 @@
 
         // JAXB will expect a namespace if we try to unmarshall, but some error responses don't include
         // a namespace. In lieu of writing a SAXFilter to apply a default namespace in-line, this works just as well.
-        SAXReader saxReader = new SAXReader();
+        // Replace DOM4J (SAXReader) with JAXP DOM (DocumentBuilderFactory) for XML parsing
+        javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+        
+        javax.xml.parsers.DocumentBuilder dbf = dbf.newDocumentBuilder();
         saxReader.setFeature("http://apache.org/xml/features/disallow-doctype-decl", true);
         saxReader.setFeature("http://apache.org/xml/features/external-general-entities", false);
         saxReader.setFeature("http://xml.org/sax/features/external-parameter-entities", false);
@@ -135,4 +138,4 @@
         log.debug("Error: {}, message: {}, requestId: {}", new Object[]{code, message, requestId});
         return new S3Exception(message, statusCode, code, requestId);
     }
-}
+}