--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/JDomParser.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXBuilder__TO__InputFactory/JDomParser.java
@@ -22,7 +22,8 @@
 
     public List<Element> getAllTitles() {
         try {
-            SAXBuilder builder = new SAXBuilder();
+            javax.xml.stream.XMLInputFactory builder = javax.xml.stream.XMLInputFactory.newFactory();
+            
             Document doc = builder.build(this.getFile());
             Element tutorials = doc.getRootElement();
             List<Element> titles = tutorials.getChildren("tutorial");
@@ -36,7 +37,7 @@
 
     public Element getNodeById(String id) {
         try {
-            SAXBuilder builder = new SAXBuilder();
+            javax.xml.stream.XMLInputFactory builder = javax.xml.stream.XMLInputFactory.newFactory();
             Document document = (Document) builder.build(file);
             String filter = "//*[@tutId='" + id + "']";
             XPathFactory xFactory = XPathFactory.instance();
@@ -58,4 +59,4 @@
         this.file = file;
     }
 
-}
+}