--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/PluginTestUsername.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXBuilder__TO__InputFactory/PluginTestUsername.java
@@ -47,7 +47,8 @@
 	@Before
 	public void setUp() throws Exception {
 
-		SAXBuilder builder = new SAXBuilder();
+		javax.xml.stream.XMLInputFactory builder = javax.xml.stream.XMLInputFactory.newFactory();
+
 		try {
 			document = builder.build(new File("./testData.xml")
 					.getCanonicalPath());
@@ -193,4 +194,4 @@
 										mainViewId));
 	}
 
-}
+}