--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/UnitTestLogin.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXBuilder__TO__InputFactory/UnitTestLogin.java
@@ -29,7 +29,8 @@
 	public void setUp() {
 		pw = new ProxyWrapper();
 
-		SAXBuilder builder = new SAXBuilder();
+		javax.xml.stream.XMLInputFactory builder = javax.xml.stream.XMLInputFactory.newFactory();
+
 		try {
 			document = builder.build(new File("./testData.xml")
 					.getCanonicalPath());
@@ -76,4 +77,4 @@
 
 	}
 
-}
+}