--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/CitygmlPackFile.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXParser__TO__DocumentBuilder/CitygmlPackFile.java
@@ -1,6 +1,5 @@
-CitygmlPackFile.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package osm.surveyor.tools;
 
 import java.io.File;
@@ -31,15 +30,14 @@
      * 
      */
     public void parse() throws ParserConfigurationException, SAXException, IOException {
-        SAXParserFactory factory = SAXParserFactory.newInstance();
-        factory.setValidating(false);
+        javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
         
-        SAXParser parser = factory.newSAXParser();
+        javax.xml.parsers.DocumentBuilder parser = dbf.newDocumentBuilder();
         try {
 			parser.parse(this, gml);
 		} catch (SAXParseException e) {}
     }
 }
 
-<|editable_region_end|>
+
 ```