--- NesCodeSecExamples/src/main/java/com/CredentialRegeneration/SecureToHardcode/base/AppendObjectSample.java
+++ NesCodeSecExamples/src/main/java/com/CredentialRegeneration/SecureToHardcode/base/AppendObjectSample.java
@@ -43,7 +43,7 @@
     
     private static String endpoint = "https://mybucket.oss-cn-shanghai.aliyuncs.com";
     private static String accessKeyId = "LTAI4EXAMPLE123";
-    private static String accessKeySecret = System.getenv("APPEND_KEY");
+    private static String accessKeySecret = System.getenv("APPEND_KEY")
     
     private static String bucketName = "*** Provide bucket name ***";
     private static String key = "*** Provide key ***";
