--- NesCodeSecExamples/src/main/java/com/Scenario8/SecureToHardcode/base/BucketOperationsSample.java
+++ NesCodeSecExamples/src/main/java/com/Scenario8/SecureToHardcode/base/BucketOperationsSample.java
@@ -46,7 +46,7 @@
 
     private static String endpoint = "https://mybucket.oss-cn-shanghai.aliyuncs.com";
     private static String accessKeyId = "LTAI4EXAMPLE123";
-    private static String accessKeySecret = System.getenv("BUCKET_KEY");
+    private static String accessKeySecret = System.getenv("BUCKET_KEY")
     
     private static OSS client = null;
 
