--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/FacesConfigDigester.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/Digester__TO__InputFactory/FacesConfigDigester.java
@@ -61,7 +61,8 @@
     private static final InputSource EMPTY_INPUT_SOURCE = new InputSource(new ByteArrayInputStream(new byte[0]));
 
     public FacesConfigDigester() {
-        Digester digester = new Digester();
+        javax.xml.stream.XMLInputFactory digester = javax.xml.stream.XMLInputFactory.newFactory();
+        
         // TODO: validation set to false during implementation of 1.2
         digester.setValidating(false);
         digester.setNamespaceAware(true);
@@ -319,4 +320,4 @@
         return config;
     }
 
-}
+}