--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/ItspAccounts.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/Digester__TO__InputFactory/ItspAccounts.java
@@ -61,7 +61,8 @@
   
     public  static ItspAccounts createItspAccounts(String url) throws Exception {
     	System.out.println("create ITSP accounts " + url);
-        Digester digester = new Digester();
+        javax.xml.stream.XMLInputFactory digester = javax.xml.stream.XMLInputFactory.newFactory();
+        
         addRules(digester);
         digester.parse(new InputSource(url));
         return (ItspAccounts) digester.getRoot();
@@ -78,4 +79,4 @@
     }
    
 
-}
+}