--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/XMLTextConceptDigester.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/Digester__TO__InputFactory/XMLTextConceptDigester.java
@@ -34,7 +34,8 @@
   public Results parseAlchemyXML(InputStream xmlReader) throws IOException, SAXException,
           URISyntaxException {
 
-    Digester digester = new Digester();
+    javax.xml.stream.XMLInputFactory digester = javax.xml.stream.XMLInputFactory.newFactory();
+
     digester.setValidating(false);
 
     digester.addObjectCreate("results", ConceptResults.class);
@@ -64,4 +65,4 @@
 
     return (Results) digester.parse(xmlReader);
   }
-}
+}