--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/XxeDigester.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/Digester__TO__InputFactory/XxeDigester.java
@@ -6,7 +6,8 @@
 
 public class XxeDigester {
     public void negative() throws Exception {
-        Digester digester = new Digester();
+        javax.xml.stream.XMLInputFactory digester = javax.xml.stream.XMLInputFactory.newFactory();
+        
         <fim_suffix>
     }
 }
