--- LoginService.java
+++ LoginService.java
@@ -1,5 +1,7 @@
+
+
 package com.libraryman_api.security.services;
 
 import com.libraryman_api.member.MemberRepository;
 import com.libraryman_api.security.jwt.JwtAuthenticationHelper;
 import com.libraryman_api.security.model.LoginRequest;
@@ -28,11 +30,11 @@
         this.userDetailsService = userDetailsService;
         this.jwtHelper = jwtHelper;
         this.memberRepository = memberRepository;
     }
 
-    private LoginResponse login(LoginRequest loginRequest) {
+    public  LoginResponse login(LoginRequest loginRequest) {
         Authenticate(loginRequest.getUsername(), loginRequest.getPassword());
         UserDetails userDetails = userDetailsService.loadUserByUsername(loginRequest.getUsername());
         String token = jwtHelper.generateToken(userDetails);
         LoginResponse loginResponse = new LoginResponse(token);
         return loginResponse;
@@ -52,6 +54,8 @@
     }
 
     private boolean validatePassword(String rawPassword, String encodedPassword) {
         return jwtHelper.getPasswordEncoder().matches(rawPassword, encodedPassword);
     }
-}+
+}
+
