--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/Util.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXBuilder__TO__Digester/Util.java
@@ -62,7 +62,8 @@
 
     public static Element loadRootElement(File configFile) throws CruiseControlException {
         try {
-            SAXBuilder builder = new SAXBuilder();
+            org.apache.commons.digester3.Digester builder = new org.apache.commons.digester3.Digester();
+            
             builder.setFeature("http://apache.org/xml/features/xinclude", true);
             return builder.build(configFile).getRootElement();
         } catch (Exception e) {
@@ -76,7 +77,7 @@
 
     public static Element loadRootElement(InputStream in) throws CruiseControlException {
         try {
-            SAXBuilder builder = new SAXBuilder();
+            org.apache.commons.digester3.Digester builder = new org.apache.commons.digester3.Digester();
             return builder.build(in).getRootElement();
         } catch (Exception e) {
             throw new CruiseControlException("failed to parse configuration", e);
@@ -301,4 +302,4 @@
         }
         return string;
     }
-}
+}