--- NesCodeSecExamples/src/main/java/com/Scenario1/base/DocumentBuilder/EarthquakeUpdateJobService.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/DocumentBuilder__TO__SAXParser/EarthquakeUpdateJobService.java
@@ -102,9 +102,9 @@
       int responseCode = httpConnection.getResponseCode();
       if (responseCode == HttpURLConnection.HTTP_OK) {
         InputStream in = httpConnection.getInputStream();
-        DocumentBuilderFactory dbf =
-          DocumentBuilderFactory.newInstance();
-        DocumentBuilder db = dbf.newDocumentBuilder();
+        javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+        
+        javax.xml.parsers.SAXParser db = spf.newSAXParser();
 
         // Parse the earthquake feed.
         Document dom = db.parse(in);
