--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/JDomParser.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXBuilder__TO__Digester/JDomParser.java
@@ -25 +25,2 @@
-            SAXBuilder builder = new SAXBuilder();
+            org.apache.commons.digester3.Digester builder = new org.apache.commons.digester3.Digester();
+            
@@ -39 +40 @@
-            SAXBuilder builder = new SAXBuilder();
+            org.apache.commons.digester3.Digester builder = new org.apache.commons.digester3.Digester();
@@ -61 +62 @@
-}
+}