--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/PluginTestUsername.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXBuilder__TO__Digester/PluginTestUsername.java
@@ -1,6 +1,5 @@
-PluginTestUsername.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package registration.plugintest;
 
 import static org.junit.Assert.*;
@@ -50,7 +49,8 @@
 	@Before
 	public void setUp() throws Exception {
 
-		SAXBuilder builder = new SAXBuilder();
+		org.apache.commons.digester3.Digester builder = new org.apache.commons.digester3.Digester();
+
 		try {
 			document = builder.build(new File("./testData.xml")
 					.getCanonicalPath());
@@ -198,5 +198,5 @@
 
 }
 
-<|editable_region_end|>
+
 ```