--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXBuilder/UnitTestLogin.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/SAXBuilder__TO__Digester/UnitTestLogin.java
@@ -1,6 +1,5 @@
-UnitTestLogin.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package login;
 
 import java.io.File;
@@ -32,7 +31,8 @@
 	public void setUp() {
 		pw = new ProxyWrapper();
 
-		SAXBuilder builder = new SAXBuilder();
+		org.apache.commons.digester3.Digester builder = new org.apache.commons.digester3.Digester();
+
 		try {
 			document = builder.build(new File("./testData.xml")
 					.getCanonicalPath());
@@ -81,5 +81,5 @@
 
 }
 
-<|editable_region_end|>
+
 ```