--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/FacesConfigDigester.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/Digester__TO__DocumentBuilder/FacesConfigDigester.java
@@ -61,7 +61,9 @@
     private static final InputSource EMPTY_INPUT_SOURCE = new InputSource(new ByteArrayInputStream(new byte[0]));
 
     public FacesConfigDigester() {
-        Digester digester = new Digester();
+        javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+        
+        javax.xml.parsers.DocumentBuilder digester = dbf.newDocumentBuilder();
         // TODO: validation set to false during implementation of 1.2
         digester.setValidating(false);
         digester.setNamespaceAware(true);
@@ -125,4 +127,4 @@
         return config;
     }
 
-}
+}