--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/GoogleGeocoderImpl.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/Digester__TO__DocumentBuilder/GoogleGeocoderImpl.java
@@ -75,7 +75,10 @@
 
   private Digester createDigester() {
 
-    Digester digester = new Digester();
+    javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+
+
+    javax.xml.parsers.DocumentBuilder digester = dbf.newDocumentBuilder();
 
     Class<?>[] dType = {Double.class};
 
@@ -110,4 +113,4 @@
       throw new IllegalStateException(e);
     }
   }
-}
+}