--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/hwxml.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__SAXParser/hwxml.java
@@ -1,6 +1,5 @@
-hwxml.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 /**
  * EGroupware - Notifications Java Desktop App
  *
@@ -86,7 +85,9 @@
     {
         if( !this.newfile )
         {
-            XMLInputFactory inputFactory = XMLInputFactory.newInstance();
+            javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+            
+            javax.xml.parsers.SAXParser inputFactory = spf.newSAXParser();
 			
 			InputStream in = null;
 			
@@ -178,5 +179,5 @@
     }
 }
 
-<|editable_region_end|>
+
 ```