--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/StaxParser(2).java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__SAXParser/StaxParser(2).java
@@ -1,6 +1,5 @@
-StaxParser(2).java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package com.github.mfatihercik.dsb.xml;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
@@ -33,7 +32,9 @@
 
     @Override
     protected void parseInputStream(Reader reader) {
-        XMLInputFactory factory = XMLInputFactory.newInstance();
+        javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+        
+        javax.xml.parsers.SAXParser factory = spf.newSAXParser();
         try {
             XMLStreamReader streamReader = factory.createXMLStreamReader(reader);
             processReader(streamReader);
@@ -46,7 +47,8 @@
 
     @Override
     protected void parseInputStream(InputStream inputStream) {
-        XMLInputFactory factory = XMLInputFactory.newInstance();
+        javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+        javax.xml.parsers.SAXParser factory = spf.newSAXParser();
         try {
             XMLStreamReader streamReader = factory.createXMLStreamReader(inputStream);
             processReader(streamReader);
@@ -213,5 +215,5 @@
 
 }
 
-<|editable_region_end|>
+
 ```