--- NesCodeSecExamples/src/main/java/com/Scenario1/base/DocumentBuilder/DomToGroovy.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/DocumentBuilder__TO__InputFactory/DomToGroovy.java
@@ -1,6 +1,5 @@
-DomToGroovy.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 /**
  *  Licensed to the Apache Software Foundation (ASF) under one
  *  or more contributor license agreements.  See the NOTICE file
@@ -112,16 +111,13 @@
     }
 
     public static Document parse(final Reader input) throws Exception {
-        DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
-        factory.setNamespaceAware(true);
-        DocumentBuilder builder = factory.newDocumentBuilder();
+        javax.xml.stream.XMLInputFactory builder = javax.xml.stream.XMLInputFactory.newFactory();
+        
         return builder.parse(new InputSource(input));
     }
 
     public static Document parse(final InputStream input) throws Exception {
-        DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
-        factory.setNamespaceAware(true);
-        DocumentBuilder builder = factory.newDocumentBuilder();
+        javax.xml.stream.XMLInputFactory builder = javax.xml.stream.XMLInputFactory.newFactory();
         return builder.parse(new InputSource(input));
     }
 
@@ -401,5 +397,5 @@
     }
 }
 
-<|editable_region_end|>
+
 ```