--- NesCodeSecExamples/src/main/java/com/Scenario8/base/UnsplashGallery.java
+++ NesCodeSecExamples/src/main/java/com/Scenario8/input_event/BucketCnameSample/UnsplashGallery.java
@@ -13,7 +13,7 @@
 
 public class UnsplashGallery extends Application {
 
-    private static final String ACCESS_KEY = "rSzPecnF8pvlW3t1K6pUjJGPFXyWCg-Q0tRvBw7Gaj0"; // 🔑 Unsplash API key
+    private static final String ACCESS_KEY = 
     private static final String API_URL = "https://api.unsplash.com/photos/random?count=10&client_id=" + ACCESS_KEY;
 
     @Override
