--- NesCodeSecExamples/src/main/java/com/Scenario1/base/DocumentBuilder/XxeDocumentBuilderFactory.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/DocumentBuilder__TO__SAXBuilder/XxeDocumentBuilderFactory.java
@@ -1,6 +1,5 @@
-XxeDocumentBuilderFactory.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 
 package examples;
 
@@ -19,10 +18,11 @@
 public class XxeDocumentBuilderFactory {
 
     public void negative() throws SAXException, DocumentException, ParserConfigurationException, IOException {
-        DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+        org.jdom2.input.SAXBuilder factory = new org.jdom2.input.SAXBuilder();
+        
         
         Document document = builder.parse(in);
     }
 }
-<|editable_region_end|>
+
 ```