--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/DefaultSAXPrologProcessRegistry.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXParser__TO__SAXBuilder/DefaultSAXPrologProcessRegistry.java
@@ -42,7 +42,9 @@
 		//Element cpElement;
 
 		try {
-			SAXParser parser = SAXParserFactory.newInstance().newSAXParser();
+			// Replace SAX (SAXParserFactory) with JDOM2 (SAXBuilder) for XML parsing
+			org.jdom2.input.SAXBuilder saxBuilder = new org.jdom2.input.SAXBuilder();
+			
 			//parser.setProperty("http://xml.org/sax/features/namespaces", new Boolean(true));
 			//boolean b = parser.isNamespaceAware();
 			parser.parse(new InputSource(reader),new RegistryHandler());
@@ -141,6 +143,4 @@
 
 	
 
-}
-
-
+}