--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/PhysicsEditorShapeLibrary.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXParser__TO__SAXBuilder/PhysicsEditorShapeLibrary.java
@@ -106 +106,2 @@
-        SAXParserFactory factory = SAXParserFactory.newInstance();
+        org.jdom2.input.SAXBuilder factory = new org.jdom2.input.SAXBuilder();
+        
@@ -230,2 +231 @@
-}
-
+}