--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXParser/ReadXMLFileSAX.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXParser__TO__SAXBuilder/ReadXMLFileSAX.java
@@ -16,2 +16,2 @@
-            SAXParserFactory factory = SAXParserFactory.newInstance();
-            SAXParser saxParser = factory.newSAXParser();
+            org.jdom2.input.SAXBuilder saxParser = new org.jdom2.input.SAXBuilder();
+            
@@ -33 +33 @@
-}
+}