--- NesCodeSecExamples/src/main/java/com/Scenario1/base/SAXReader/PRConfig.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/SAXReader__TO__SAXParser/PRConfig.java
@@ -1,4 +1,3 @@
-
 package com.pega.gcs.logviewer.systemstate.model;
 
 import java.io.StringReader;
@@ -100,7 +99,12 @@
 
         if (xmlContent != null) {
 
-            SAXReader saxReader = new SAXReader();
+            // Replace DOM4J (SAXReader) with SAX (SAXParserFactory) for XML parsing
+
+            javax.xml.parsers.SAXParserFactory spf = javax.xml.parsers.SAXParserFactory.newInstance();
+
+
+            javax.xml.parsers.SAXParser spf = spf.newSAXParser();
 
             StringReader stringReader = new StringReader(xmlContent);
 
@@ -125,4 +129,4 @@
         }
 
     }
-}
+}