--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/ContactData.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__DocumentBuilder/ContactData.java
@@ -1,6 +1,5 @@
-ContactData.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 package sample.datamodel;
 
 import javafx.collections.FXCollections;
@@ -60,7 +59,9 @@
     public void loadContacts() {
         try {
             // First, create a new XMLInputFactory
-            XMLInputFactory inputFactory = XMLInputFactory.newInstance();
+            javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+            
+            javax.xml.parsers.DocumentBuilder inputFactory = dbf.newDocumentBuilder();
             // Setup a new eventReader
             InputStream in = new FileInputStream(CONTACTS_FILE);
             XMLEventReader eventReader = inputFactory.createXMLEventReader(in);
@@ -205,5 +206,5 @@
     }
 
 }
-<|editable_region_end|>
+
 ```