--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/MetricsDefManager.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__DocumentBuilder/MetricsDefManager.java
@@ -1,6 +1,5 @@
-MetricsDefManager.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 /*
  * Copyright 2015, Yahoo Inc.
  * Copyrights licensed under the Apache License.
@@ -150,7 +149,9 @@
 		if (in == null)return groups;
 		
 	    XMLStreamReader reader = null;
-	    XMLInputFactory inputFactory = XMLInputFactory.newInstance();//will be used only once at startup time
+	    javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+	    
+	    javax.xml.parsers.DocumentBuilder inputFactory = dbf.newDocumentBuilder();//will be used only once at startup time
 		try
 		{
 		  reader = inputFactory.createXMLStreamReader(new java.io.InputStreamReader(in));
@@ -339,5 +340,5 @@
 
 }
 
-<|editable_region_end|>
+
 ```