--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/XxeInputFactory.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__DocumentBuilder/XxeInputFactory.java
@@ -1,6 +1,5 @@
-XxeInputFactory.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 
 package examples;
 
@@ -15,8 +14,10 @@
 public class XxeInputFactory {
 
     public void negative(InputStream is) throws SAXException, IOException, XMLStreamException {
-        XMLInputFactory factory = XMLInputFactory.newInstance();
+        javax.xml.parsers.DocumentBuilderFactory dbf = javax.xml.parsers.DocumentBuilderFactory.newInstance();
+        
+        javax.xml.parsers.DocumentBuilder factory = dbf.newDocumentBuilder();
     }
 }
-<|editable_region_end|>
+
 ```