--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/GoogleGeocoderImpl.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/Digester__TO__SAXBuilder/GoogleGeocoderImpl.java
@@ -75,7 +75,8 @@
 
   private Digester createDigester() {
 
-    Digester digester = new Digester();
+    org.jdom2.input.SAXBuilder digester = new org.jdom2.input.SAXBuilder();
+
 
     Class<?>[] dType = {Double.class};
 
@@ -110,4 +111,4 @@
       throw new IllegalStateException(e);
     }
   }
-}
+}