--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/GoogleGeocoderImpl.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/Digester__TO__SAXBuilder/GoogleGeocoderImpl.java
@@ -1,6 +1,5 @@
-GoogleGeocoderImpl.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 /**
  * Copyright (C) 2011 Brian Ferris <bdferris@onebusaway.org>
  *
@@ -78,7 +77,8 @@
 
   private Digester createDigester() {
 
-    Digester digester = new Digester();
+    org.jdom2.input.SAXBuilder digester = new org.jdom2.input.SAXBuilder();
+
 
     Class<?>[] dType = {Double.class};
 
@@ -115,5 +115,5 @@
   }
 }
 
-<|editable_region_end|>
+
 ```