--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/ItspAccounts.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/Digester__TO__SAXBuilder/ItspAccounts.java
@@ -61,7 +61,8 @@
   
     public  static ItspAccounts createItspAccounts(String url) throws Exception {
     	System.out.println("create ITSP accounts " + url);
-        Digester digester = new Digester();
+        org.jdom2.input.SAXBuilder digester = new org.jdom2.input.SAXBuilder();
+        
         addRules(digester);
         digester.parse(new InputSource(url));
         return (ItspAccounts) digester.getRoot();
@@ -78,4 +79,4 @@
     }
    
 
-}
+}