--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/ParseImporterXML.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/Digester__TO__SAXBuilder/ParseImporterXML.java
@@ -15,7 +15,8 @@
 
 	public Importer parseXML(String xml, String dir) throws IOException,SAXException {
 
-		Digester digester = new Digester();
+		org.jdom2.input.SAXBuilder digester = new org.jdom2.input.SAXBuilder();
+
 		digester.setValidating(false);
 
 		digester.addObjectCreate("importer", Importer.class);
@@ -37,4 +38,4 @@
 		return (Importer) digester.parse(importerFile);
 	}
 
-}
+}