--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/Test03.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/Digester__TO__SAXBuilder/Test03.java
@@ -10,7 +10,8 @@
   public static void main(String[] args) {
     String path = System.getProperty("user.dir") + File.separator  + "etc";
     File file = new File(path, "employee2.xml");
-    Digester digester = new Digester();
+    org.jdom2.input.SAXBuilder digester = new org.jdom2.input.SAXBuilder();
+    
     digester.addRuleSet(new EmployeeRuleSet());
     try {
       Employee employee = (Employee) digester.parse(file);
@@ -31,5 +32,4 @@
       e.printStackTrace();
     }
   }
-}
-  
+}