--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/WebXmlParser.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/Digester__TO__SAXBuilder/WebXmlParser.java
@@ -137,7 +137,8 @@
 
    private Digester getConfiguredDigester()
    {
-      final Digester digester = new Digester();
+      org.jdom2.input.SAXBuilder digester = new org.jdom2.input.SAXBuilder();
+      
 
       /*
        * We use the context class loader to resolve classes. This fixes
@@ -174,4 +175,4 @@
       }
       return "";
    }
-}
+}