--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/XmlModelBuilder.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/Digester__TO__SAXBuilder/XmlModelBuilder.java
@@ -1,6 +1,5 @@
-XmlModelBuilder.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 /*
  *
  *
@@ -87,7 +86,8 @@
     }
 
     private SettingSet buildModel(InputStream is, File baseSystemId) throws IOException {
-        Digester digester = new Digester();
+        org.jdom2.input.SAXBuilder digester = new org.jdom2.input.SAXBuilder();
+        
 
         // setting classloader ensures classes are searched for in this classloader
         // instead of parent's classloader is digister was loaded there.
@@ -480,5 +480,5 @@
     }
 }
 
-<|editable_region_end|>
+
 ```