--- NesCodeSecExamples/src/main/java/com/Scenario1/base/Digester/XxeDigester.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/Digester__TO__SAXBuilder/XxeDigester.java
@@ -6,7 +6,8 @@
 
 public class XxeDigester {
     public void negative() throws Exception {
-        Digester digester = new Digester();
+        org.jdom2.input.SAXBuilder digester = new org.jdom2.input.SAXBuilder();
+        
         <fim_suffix>
     }
 }
