--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/ForeignMetadataImportServiceBean.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__SAXReader/ForeignMetadataImportServiceBean.java
@@ -1,6 +1,5 @@
-ForeignMetadataImportServiceBean.java
-```<|start_of_file|>
-<|editable_region_start|>
+```
+
 
 package edu.harvard.iq.dataverse.metadataimport;
 
@@ -73,7 +72,8 @@
         
         try {
             reader = new StringReader(xmlToParse);
-            XMLInputFactory xmlFactory = XmlUtil.getSecureXMLInputFactory();
+            org.dom4j.io.SAXReader xmlFactory = new org.dom4j.io.SAXReader();
+            
             xmlr =  xmlFactory.createXMLStreamReader(reader);
             processXML(xmlr, mappingSupported, datasetVersion);
         
@@ -100,7 +100,7 @@
         
         try {
             in = new FileInputStream(xmlFile);
-            XMLInputFactory xmlFactory = XmlUtil.getSecureXMLInputFactory();
+            org.dom4j.io.SAXReader xmlFactory = new org.dom4j.io.SAXReader();
             xmlr =  xmlFactory.createXMLStreamReader(in);
             processXML(xmlr, mappingSupported, datasetVersion);
         } catch (FileNotFoundException ex) {
@@ -348,5 +348,5 @@
     
 }
 
-<|editable_region_end|>
+
 ```