--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/MetricsDefManager.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/migrate_full/InputFactory__TO__SAXReader/MetricsDefManager.java
@@ -150 +150,3 @@
-	    XMLInputFactory inputFactory = XMLInputFactory.newInstance();//will be used only once at startup time
+	    org.dom4j.io.SAXReader inputFactory = new org.dom4j.io.SAXReader();
+	    
+//will be used only once at startup time
@@ -337 +339 @@
-}
+}