--- NesCodeSecExamples/src/main/java/com/Scenario1/base/InputFactory/StaxParser.java
+++ NesCodeSecExamples/src/main/java/com/Scenario1/input_excerpt/InputFactory__TO__SAXReader/StaxParser.java
@@ -30,7 +30,8 @@
 
     @Override
     protected void parseInputStream(Reader reader) {
-        XMLInputFactory factory = XMLInputFactory.newInstance();
+        org.dom4j.io.SAXReader factory = new org.dom4j.io.SAXReader();
+        
         try {
             XMLStreamReader streamReader = factory.createXMLStreamReader(reader);
             processReader(streamReader);
@@ -43,7 +44,7 @@
 
     @Override
     protected void parseInputStream(InputStream inputStream) {
-        XMLInputFactory factory = XMLInputFactory.newInstance();
+        org.dom4j.io.SAXReader factory = new org.dom4j.io.SAXReader();
         try {
             XMLStreamReader streamReader = factory.createXMLStreamReader(inputStream);
             processReader(streamReader);
@@ -208,4 +209,4 @@
     }
 
 
-}
+}